<!DOCTYPE html>



  


<html class="theme-next muse use-motion" lang="">
<head>
  <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
<meta name="theme-color" content="#222">









<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />
















  
  
  <link href="/lib/fancybox/source/jquery.fancybox.css?v=2.1.5" rel="stylesheet" type="text/css" />







<link href="/lib/font-awesome/css/font-awesome.min.css?v=4.6.2" rel="stylesheet" type="text/css" />

<link href="/css/main.css?v=5.1.4" rel="stylesheet" type="text/css" />


  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png?v=5.1.4">


  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png?v=5.1.4">


  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png?v=5.1.4">


  <link rel="mask-icon" href="/images/logo.svg?v=5.1.4" color="#222">





  <meta name="keywords" content="Hexo, NexT" />










<meta property="og:type" content="website">
<meta property="og:title" content="Hexo">
<meta property="og:url" content="http://yoursite.com/index.html">
<meta property="og:site_name" content="Hexo">
<meta property="og:locale" content="default">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Hexo">



<script type="text/javascript" id="hexo.configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Muse',
    version: '5.1.4',
    sidebar: {"position":"left","display":"post","offset":12,"b2t":false,"scrollpercent":false,"onmobile":false},
    fancybox: true,
    tabs: true,
    motion: {"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},
    duoshuo: {
      userId: '0',
      author: 'Author'
    },
    algolia: {
      applicationID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    }
  };
</script>



  <link rel="canonical" href="http://yoursite.com/"/>





  <title>Hexo</title>
  








</head>

<body itemscope itemtype="http://schema.org/WebPage" lang="default">

  
  
    
  

  <div class="container sidebar-position-left 
  page-home">
    <div class="headband"></div>

    <header id="header" class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-wrapper">
  <div class="site-meta ">
    

    <div class="custom-logo-site-title">
      <a href="/"  class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Hexo</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
      
        <p class="site-subtitle"></p>
      
  </div>

  <div class="site-nav-toggle">
    <button>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
    </button>
  </div>
</div>

<nav class="site-nav">
  

  
    <ul id="menu" class="menu">
      
        
        <li class="menu-item menu-item-home">
          <a href="/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-home"></i> <br />
            
            Home
          </a>
        </li>
      
        
        <li class="menu-item menu-item-archives">
          <a href="/archives/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-archive"></i> <br />
            
            Archives
          </a>
        </li>
      

      
    </ul>
  

  
</nav>



 </div>
    </header>

    <main id="main" class="main">
      <div class="main-inner">
        <div class="content-wrap">
          <div id="content" class="content">
            
  <section id="posts" class="posts-expand">
    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/15/SIMCA_in_action/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/15/SIMCA_in_action/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-15T23:31:43+08:00">
                2018-04-15
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="SIMCA官方的NMR代谢组学教程实战"><a href="#SIMCA官方的NMR代谢组学教程实战" class="headerlink" title="SIMCA官方的NMR代谢组学教程实战"></a>SIMCA官方的NMR代谢组学教程实战</h1><h1 id="用正交偏最小二乘法判别分析处理核磁共振代谢组学数据"><a href="#用正交偏最小二乘法判别分析处理核磁共振代谢组学数据" class="headerlink" title="用正交偏最小二乘法判别分析处理核磁共振代谢组学数据"></a>用正交偏最小二乘法判别分析处理核磁共振代谢组学数据</h1><p>华中科技大学同济医学院公共卫生学院 夏鹏飞<br><a href="mailto:U201412842@hust.edu.cn" target="_blank" rel="noopener">U201412842@hust.edu.cn</a><br><a href="https://github.com/XIAPengfei1996" target="_blank" rel="noopener">https://github.com/XIAPengfei1996</a></p>
<h2 id="代谢组学数据分析简介"><a href="#代谢组学数据分析简介" class="headerlink" title="代谢组学数据分析简介"></a>代谢组学数据分析简介</h2><p>代谢组学几乎完全不涉及生物信息学最核心的序列比对，包括武汉中科院数物所波谱国重实验室等主流科研机构都是利用化学计量学和多元统计分析方法，对通过核磁共振（NMR）、液相色谱质谱联用（LC-MS）、气相色谱质谱联用（GC-MS）等手段获取的小分子代谢物（相对分子质量MW小于1000）的定量数据进行处理。除此之外只有一些商业公司在做外包服务，因此相比基因组、转录组、蛋白质组显得非常小众。</p>
<p>目前在代谢组学中运用较多的多元统计分析方法，包括主成分分析（Principal component analysis, PCA）、层次聚类分析（Hierarchical clustering analysis, HCA）、非线性映射（Nonlinear innuendo analysis, NLM）、分级聚类法（hierarchical cluster analysis, HCA）等非监督分类方法，以及K最邻近法（K-nearest neighbor classification method, K-NN）、偏最小二乘法-判别分析（Partial least squares projections to latent structures - discriminant analysis, PLS-DA）、正交偏最小二乘法-判别分析（Orthogonal projections to latent structures - discriminant analysis, OPLS-DA）、人工神经网络（Artificial Neural Network, ANN）、支持向量机（Support Vector Machine, SVM）等监督分类方法。篇幅所限，有关这些方法的具体内容和意义请参考相关文献。</p>
<p>代谢组学数据分析最常用的软件SIMCA是1987年由瑞典Umetrics公司研究开发的一款多元变量统计分析软件。官方网站随便一个课程视频都是一、两千美刀，其他学习资源目前也都限于各生物公司的培训资料，同样价格不菲。这里我利用SIMCA官方的教程和数据操作一遍完整的流程以飨读者。</p>
<h2 id="下载并安装软件"><a href="#下载并安装软件" class="headerlink" title="下载并安装软件"></a>下载并安装软件</h2><ul>
<li><a href="http://landing.umetrics.com/downloads-simca" target="_blank" rel="noopener">官方网站的下载页面</a>提供了30天免费试用版SIMCA，当前更新到第15版；下载解压得到.exe格式文件，安装后即可在Windows系统上运行。由于之前为了给转录组入门实战数据腾位置，我删掉了苹果机上的Windows双系统，这次我将临时在Parallels Desktop 13试用版虚拟机上运行Windows 10系统来进行操作。</li>
</ul>
<h2 id="教程及数据"><a href="#教程及数据" class="headerlink" title="教程及数据"></a>教程及数据</h2><ul>
<li><a href="http://landing.umetrics.com/downloads-simca" target="_blank" rel="noopener">官网下载页</a>虽然也提供了官方教材<a href="https://github.com/XIAPengfei1996/Shia/raw/master/SIMCA/Analysing_NMR_metabolomics_data_using_OPLS-DA.pdf" target="_blank" rel="noopener">Analysing_NMR_metabolomics_data_using_OPLS-DA.pdf</a>，但是和数据<a href="https://github.com/XIAPengfei1996/Shia/raw/master/SIMCA/NMR%20METABOLOMICS.xls" target="_blank" rel="noopener">NMR_METABOLOMICS.xls</a>一样都非常不便下载，不过好在网上随便搜索一下也能找到很多流传的资源。方便起见，我上传到了我的GitHub里面，点击文件标题名就可以直接下载。</li>
</ul>
<h2 id="数据信息"><a href="#数据信息" class="headerlink" title="数据信息"></a>数据信息</h2><p>该研究以杨树的文库中编码功能未知的转录因子MYB的基因PttMYB76为例，比较MYB76杨树与野生型杨树生长过程的代谢组学特征差异。实验材料包括来自MYB76杨树（编号为Ai, Bi, Ci）和野生型杨树（编号为Di, Ei, Fi）两组共6棵树的45个样品，其中i代表生长方向，即1代表顶部，8代表底部。加上12组重复观测值，数据集中总共有57组观测值，655种变量。由于这里我们主要目的是做数据分析，因此对实验细节不加赘述，有需要者可以参考原文：<br><a href="https://eurekamag.com/pdf.php?pdf=004019559" target="_blank" rel="noopener">S. Wiklund et.al A new metabonomic strategy for analysing the growth process of the poplar tree. Plant Biotechnology Journal 2005 3 pp 353-362</a><br><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/1_excel.png" alt="1_excel"></p>
<h2 id="数据导入"><a href="#数据导入" class="headerlink" title="数据导入"></a>数据导入</h2><p>打开菜单栏中的<code>File</code>，点击<code>New</code>选项中的<code>Regular project</code>弹出浏览界面，选择点击下载好的数据集<code>NMR_METABOLOMICS.xls</code>文件导入数据。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/2_file.png" alt="2_file"></p>
<p>这里还需要按照软件要求调整数据格式，打开菜单栏中的<code>Edit</code>，点击<code>Transpose</code>将表格翻转90度；</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/3_transpose.png" alt="3_transpose"></p>
<p>在打开菜单栏中的<code>Home</code>，依次选择第一行后点击<code>Primary variable ID</code>，选择第二列后点击<code>Secondary observation ID</code>；最后点击右上角的<code>Finish import</code>，保存后即完成数据导入。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/4_import.png" alt="4_import"></p>
<h2 id="主成分分析"><a href="#主成分分析" class="headerlink" title="主成分分析"></a>主成分分析</h2><p>打开新建的模型M1，点击模型界面左上角的<code>Workset…</code>，选择<code>Scale</code>界面，点击左下角<code>Find Primary ID…</code>右侧的三角形，选择第一行的<code>Select all</code>（或者<code>Ctrl</code>+<code>A</code>）全选变量名；然后点击右侧<code>Type</code>项中下拉按钮，选择<code>Par</code>选项后，点击下方的<code>Set all</code>，对所有变量值进行Pareto标度化。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/6_scale.png" alt="6_scale"></p>
<p>再选择<code>Observations</code>界面，长按<code>Ctrl</code>键，选出A、B、C三棵MYB76杨树对应观测值，在<code>Set class</code>栏中输入2，点击<code>Set</code>将MYB76杨树观测值编为2组；同理将D、E、F三棵野生型杨树对应观测值编为1组；再在下方的<code>Model typr</code>栏中选择<code>PCA-X</code>，点击<code>确定</code>完成设置。这里要求处理组样本需要具有较高的类别编号，对照样本需要具有较低的类别编号，这样做会有利于后面的直接解释。 </p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/7_observations.png" alt="7_observations"></p>
<p>选择新建的M1模型，点击上方工具栏中的<code>Autofit</code>自动拟合最佳模型。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/8_autofit.png" alt="8_autofit"></p>
<p>生成的模型产生的8个主成分，这里我们看到第8主个成分增加后，模型预测性Q2没有明显增加，说明这个主成分对于模型的贡献以及非常微小。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/9_fit.png" alt="9_fit"></p>
<p>为了便于分析，这里我们只选取前3个主成分，可以通过点击<code>Autofit</code>右侧的<code>remove</code>删去多余的主成分。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/10_remove.png" alt="10_remove"></p>
<p>如果误删成分，可以点击<code>remove</code>上方的<code>add</code>恢复。（我没有点错！我只是为了给大家试错，对，没错，就是这样！）<br>最后的结果应该是这样：</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/11_component.png" alt="11_component"></p>
<p>点击上方工具栏中的<code>DModX</code>可以选择生成折线图或直方图查看离群值。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/12_DModX.png" alt="12_DModX"></p>
<p>点击上方工具栏中的<code>Scores</code>可以选择<code>Scatter</code>生成以第一主成分t1和第二主成分t2分别为横纵坐标的t1-t2得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/13_scoreplott1t2.png" alt="13_scoreplott1t2"></p>
<p>也可以选择<code>3D</code>生成分别以三个主成分为坐标的三维得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/14_scoreplot3d.png" alt="14_scoreplot3d"></p>
<p>这里两组样本数据并没有得到很好的区分，我们需要在看一下t2-t3得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/15_scoreplott2t3.png" alt="15_scoreplott2t3"></p>
<p>实现这个得分图需要点击工具栏中的<code>Scores</code>选项下最右侧的图标，或者右键点击之前生成的得分图，点击第二个图标，进入设置界面。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/16_dialog.png" alt="16_dialog">、</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/17_properties.png" alt="17_properties"></p>
<p>在<code>Scores</code>页面中<code>Selected</code>栏下，<code>X-axis</code>对应的<code>Comp</code>选择<code>2</code>，<code>Series</code>对应的<code>Comp</code>选择<code>3</code>，点击<code>确定</code>即可生成t2-t3得分图.</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/18_scores.png" alt="18_scores"></p>
<p>这里我们对比两个得分图可以明显看出，后者对两组样本数据的区分效果优于前者。</p>
<p>此外，如果仔细观察t1-t2得分图，还可以发现生长方向i值有明显地沿某一方向依次变化的趋势，表现了杨树生长过程的代谢组学特征的规律性变化。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/19_trend.png" alt="19_trend"></p>
<h2 id="正交偏最小二乘法判别分析"><a href="#正交偏最小二乘法判别分析" class="headerlink" title="正交偏最小二乘法判别分析"></a>正交偏最小二乘法判别分析</h2><p>右键点击模型M1，选择<code>New as Model 1</code>生成新的模型M2。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/20_newasmedel.png" alt="20_newasmedel"></p>
<p>选择<code>Observations</code>界面，长按<code>Ctrl</code>键，选出最后12组重复观测值，右键选择<code>Exclude</code>；再在下方的<code>Model typr</code>栏中选择<code>OPLS-DA</code>，点击<code>确定</code>完成设置。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/21_exclude.png" alt="21_exclude"></p>
<p>再次点击<code>Autofit</code>自动拟合最佳模型，得到一个1+4的OPLS模型。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/22_component.png" alt="22_component"></p>
<p>点击上方工具栏中的<code>Scores</code>，选择<code>Scatter</code>即可生成OPLS-DA得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/23_oplsda.png" alt="23_oplsda"></p>
<h2 id="主成分分析和正交偏最小二乘法判别分析的对比"><a href="#主成分分析和正交偏最小二乘法判别分析的对比" class="headerlink" title="主成分分析和正交偏最小二乘法判别分析的对比"></a>主成分分析和正交偏最小二乘法判别分析的对比</h2><p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/pca_vs_oplsda.png" alt="pca_vs_oplsda"></p>
<p>显然正交偏最小二乘法判别分析的分类效果更佳！</p>
<h2 id="解释正交偏最小二乘法判别分析模型"><a href="#解释正交偏最小二乘法判别分析模型" class="headerlink" title="解释正交偏最小二乘法判别分析模型"></a>解释正交偏最小二乘法判别分析模型</h2><p>点击上方工具栏中的<code>Loadings</code>，选择<code>Line</code>生成图片后，右键点击图片，点击<code>Properties</code>，在<code>Loadings</code>页面中选择<code>Selected</code>栏下，<code>X-axis</code>对应的<code>Item</code>选择<code>Var ID</code>，<code>Series</code>对应的<code>Item</code>选择<code>pq</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/24_loadings.png" alt="24_loadings"></p>
<p>点击<code>确定</code>生成载荷图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/25_loadingline.png" alt="25_loadingline"></p>
<p>点击上方菜单栏中的<code>Plot/List</code>，点击工具栏中的<code>Scatter</code>，在<code>Data series</code>页面中<code>Select data type</code>项选择<code>Observations and loadings</code>，<code>X-axis</code>对应的<code>Item</code>选择<code>p</code>，<code>Series</code>对应的<code>Item</code>选择<code>p(corr)</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/26_scatter.png" alt="26_scatter"></p>
<p>点击<code>确定</code>生成S-plot。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/27_splot.png" alt="27_splot"></p>
<p>标记点为可能的生物标志物。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/28_loading&amp;s.png" alt="28_loading&amp;s"></p>
<h2 id="诊断正交偏最小二乘法判别分析模型"><a href="#诊断正交偏最小二乘法判别分析模型" class="headerlink" title="诊断正交偏最小二乘法判别分析模型"></a>诊断正交偏最小二乘法判别分析模型</h2><p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/diagnostics.png" alt="diagnostics"></p>
<p><strong>模型概要</strong></p>
<p>R2X(cum)是模型所解释的X预测和正交变异之和，即0.157+0.613=0.769，也可以被解释为X总变异的76.9%；</p>
<p>R2Y(cum) 是模型所解释Y的总变异, 此处为0.977；</p>
<p>Q2(cum)衡量模型预测好坏，此处为0.941；</p>
<p><strong>预测变异 = X变异中与Y相关的部分</strong></p>
<p>A代表于X和Y之间的相关主成分的数量。如果仅使用一个响应向量，则A总是1；</p>
<p>R2X衡量X变异中与Y相关部分，这里是0.157；</p>
<p><strong>正交变异 = X变异中与Y无关的部分</strong></p>
<p>A在此对应于不相关（正交）主成分的数量。每个正交分量都被表示出来并且可以被单独解释；</p>
<p>R2X是衡量X变异中与Y不相关的部分。每个主成分的信息都被分别表示出来；</p>
<p>首行中粗体的R2X（cum）是X变异中与Y无关部分的总和，此处为0.613；</p>
<p>噪声 = 1- 0.157 – 0.613 = 0.23 = 23%</p>
<h2 id="补充"><a href="#补充" class="headerlink" title="_补充_"></a>_补充_</h2><h3 id="响应排序检验"><a href="#响应排序检验" class="headerlink" title="响应排序检验"></a>响应排序检验</h3><p>点击上方菜单栏中的<code>Analyze</code>，点击工具栏中的<code>Permutations</code>，在<code>Permutation Plot</code>页面中<code>Number of permutations to</code>项选择<code>999</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/29_permutatuons.png" alt="29_permutatuons"></p>
<p>点击<code>确定</code>生成响应排序检验示意图，通常R2Y和Q2与直线的斜率越接近水平，模型越有可能过拟合。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/30_permutatuon.png" alt="30_permutatuon"></p>
<h3 id="变量权重重要性排序"><a href="#变量权重重要性排序" class="headerlink" title="变量权重重要性排序"></a>变量权重重要性排序</h3><p>点击工具栏中的<code>VIP</code>，选择<code>VIP predictive</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/31_vip.png" alt="31_vip"></p>
<p>即直接生成变量权重重要性排序图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/32_vippredictive.png" alt="32_vippredictive"></p>
<p>右键点击图片，选择<code>Create</code>栏中<code>List</code>，可生成变量权重重要性排序表，可以选择VIP值大于1的作为感兴趣的物质进行代谢通路分析。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/33_list.png" alt="33_list"></p>
<h2 id="参考书目"><a href="#参考书目" class="headerlink" title="参考书目"></a>参考书目</h2><p>许国旺. 代谢组学:方法与应用[M]. 科学出版社, 2008.<br>贾伟. 医学代谢组学[M]. 上海科学技术出版社, 2011.</p>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/12/hello-world/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/12/hello-world/" itemprop="url">Hello World</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-12T22:24:12+08:00">
                2018-04-12
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <p>Welcome to <a href="https://hexo.io/" target="_blank" rel="noopener">Hexo</a>! This is your very first post. Check <a href="https://hexo.io/docs/" target="_blank" rel="noopener">documentation</a> for more info. If you get any problems when using Hexo, you can find the answer in <a href="https://hexo.io/docs/troubleshooting.html" target="_blank" rel="noopener">troubleshooting</a> or you can ask me on <a href="https://github.com/hexojs/hexo/issues" target="_blank" rel="noopener">GitHub</a>.</p>
<h2 id="Quick-Start"><a href="#Quick-Start" class="headerlink" title="Quick Start"></a>Quick Start</h2><h3 id="Create-a-new-post"><a href="#Create-a-new-post" class="headerlink" title="Create a new post"></a>Create a new post</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo new <span class="string">"My New Post"</span></span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/writing.html" target="_blank" rel="noopener">Writing</a></p>
<h3 id="Run-server"><a href="#Run-server" class="headerlink" title="Run server"></a>Run server</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo server</span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/server.html" target="_blank" rel="noopener">Server</a></p>
<h3 id="Generate-static-files"><a href="#Generate-static-files" class="headerlink" title="Generate static files"></a>Generate static files</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo generate</span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/generating.html" target="_blank" rel="noopener">Generating</a></p>
<h3 id="Deploy-to-remote-sites"><a href="#Deploy-to-remote-sites" class="headerlink" title="Deploy to remote sites"></a>Deploy to remote sites</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo deploy</span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/deployment.html" target="_blank" rel="noopener">Deployment</a></p>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/Methylation450K_BeadChip/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/Methylation450K_BeadChip/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/ChIP-seq/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/ChIP-seq/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/bioinfonote/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/bioinfonote/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学笔记"><a href="#生物信息学笔记" class="headerlink" title="生物信息学笔记"></a>生物信息学笔记</h1><p><a href="https://www.zhihu.com/question/20543692" target="_blank" rel="noopener">如何自学生物信息学？</a><br><a href="https://www.zhihu.com/question/65066161" target="_blank" rel="noopener">生物信息学入门需要具备什么能力？</a></p>
<p><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0167779915002735-gr1.jpg" alt="Image of Interaction Network"><br>(Source: Yugi K, et al. Trends Biotechnol, 2016)</p>
<h2 id="生物信息学-导论与方法-北京大学"><a href="#生物信息学-导论与方法-北京大学" class="headerlink" title="生物信息学: 导论与方法 | 北京大学"></a>生物信息学: 导论与方法 | 北京大学</h2><p>高歌  魏丽萍  龙漫远  <em>Michael.S.Waterman  Maynard Olson  裴钢</em></p>
<blockquote>
<p>“Essentially, all models are wrong, but some are useful.”<br>—— George E. P. Box</p>
</blockquote>
<p><a href="https://www.coursera.org/learn/sheng-wu-xin-xi-xue" target="_blank" rel="noopener">Coursera</a>  <a href="http://www.chinesemooc.org/mooc/4393" target="_blank" rel="noopener">华文慕课</a>  <a href="http://mib.cbi.pku.edu.cn" target="_blank" rel="noopener">Methods in Bioinformatics</a></p>
<ul>
<li>生信历史</li>
<li>Needleman-Wunsch全局比对算法</li>
<li>Smith-Waterman局部比对算法</li>
<li>BLAST算法</li>
<li>马尔科夫链</li>
<li>隐马尔科夫模型</li>
<li>序列回帖 <code>reads mapping</code></li>
<li>SIFT算法</li>
<li>PolyPhen算法</li>
<li>SAPRED算法</li>
<li>RNAseq回帖</li>
<li>ncRNA鉴定、差异&amp;聚类分析</li>
<li>本体论 <code>ontology</code></li>
<li>KOBAS分子通路鉴定</li>
<li>演化分析</li>
<li>各种测序原理及仪器介绍</li>
<li>各种数据库，软件、网络工具介绍</li>
</ul>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br><span class="line">43</span><br><span class="line">44</span><br><span class="line">45</span><br><span class="line">46</span><br></pre></td><td class="code"><pre><span class="line">&gt;sp|P69905|HBA_HUMAN</span><br><span class="line">MVLSPADKTN VKAAWGKVGA HAGEYGAEAL ERMFLSFPTT KTYFPHFDLS</span><br><span class="line">HGSAQVKGHG KKVADALTNA VAHVDDMPNA LSALSDLHAH KLRVDPVNFK</span><br><span class="line">LLSHCLLVTL AAHLPAEFTP AVHASLDKFL ASVSTVLTSK YR</span><br><span class="line"></span><br><span class="line">&gt;sp|P60871|HBB_HUMAN</span><br><span class="line">MVHLTPEEKS AVTALWGKVN VDEVGGEALG RLLVVYPWTQ RFFESFGDLS</span><br><span class="line">TPDAVMGNPK VKAHGKKVLG AFSDGLAHLD NLKGTFATLS ELHCDKLHVD</span><br><span class="line">PENFRLLGNV LVCVLAHHFG KEFTPPVQAA YQKVVAGVAN ALAHKYH</span><br><span class="line"></span><br><span class="line">&gt;BRCA1_HUMAN </span><br><span class="line">MDLSALRVEEVQNVINAMQKILECPICLELIKEPVSTKCDHIFCKFCMLKLLNQKKGPSQ</span><br><span class="line">CPLCKNDITKRSLQESTRFSQLVEELLKIICAFQLDTGLEYANSYNFAKKENNSPEHLKD</span><br><span class="line">EVSIIQSMGYRNRAKRLLQSEPENPSLQETSLSVQLSNLGTVRTLRTKQRIQPQKTSVYI</span><br><span class="line">ELGSDSSEDTVNKATYCSVGDQELLQITPQGTRDEISLDSAKKAACEFSETDVTNTEHHQ</span><br><span class="line">PSNNDLNTTEKRAAERHPEKYQGSSVSNLHVEPCGTNTHASSLQHENSSLLLTKDRMNVE</span><br><span class="line">KAEFCNKSKQPGLARSQHNRWAGSKETCNDRRTPSTEKKVDLNADPLCERKEWNKQKLPC</span><br><span class="line">SENPRDTEDVPWITLNSSIQKVNEWFSRSDELLGSDDSHDGESESNAKVADVLDVLNEVD</span><br><span class="line">EYSGSSEKIDLLASDPHEALICKSERVHSKSVESNIEDKIFGKTYRKKASLPNLSHVTEN</span><br><span class="line">LIIGAFVTEPQIIQERPLTNKLKRKRRPTSGLHPEDFIKKADLAVQKTPEMINQGTNQTE</span><br><span class="line">QNGQVMNITNSGHENKTKGDSIQNEKNPNPIESLEKESAFKTKAEPISSSISNMELELNI</span><br><span class="line">HNSKAPKKNRLRRKSSTRHIHALELVVSRNLSPPNCTELQIDSCSSSEEIKKKKYNQMPV</span><br><span class="line">RHSRNLQLMEGKEPATGAKKSNKPNEQTSKRHDSDTFPELKLTNAPGSFTKCSNTSELKE</span><br><span class="line">FVNPSLPREEKEEKLETVKVSNNAEDPKDLMLSGERVLQTERSVESSSISLVPGTDYGTQ</span><br><span class="line">ESISLLEVSTLGKAKTEPNKCVSQCAAFENPKGLIHGCSKDNRNDTEGFKYPLGHEVNHS</span><br><span class="line">RETSIEMEESELDAQYLQNTFKVSKRQSFAPFSNPGNAEEECATFSAHSGSLKKQSPKVT</span><br><span class="line">FECEQKEENQGKNESNIKPVQTVNITAGFPVVGQKDKPVDNAKCSIKGGSRFCLSSQFRG</span><br><span class="line">NETGLITPNKHGLLQNPYRIPPLFPIKSFVKTKCKKNLLEENFEEHSMSPEREMGNENIP</span><br><span class="line">STVSTISRNNIRENVFKEASSSNINEVGSSTNEVGSSINEIGSSDENIQAELGRNRGPKL</span><br><span class="line">NAMLRLGVLQPEVYKQSLPGSNCKHPEIKKQEYEEVVQTVNTDFSPYLISDNLEQPMGSS</span><br><span class="line">HASQVCSETPDDLLDDGEIKEDTSFAENDIKESSAVFSKSVQKGELSRSPSPFTHTHLAQ</span><br><span class="line">GYRRGAKKLESSEENLSSEDEELPCFQHLLFGKVNNIPSQSTRHSTVATECLSKNTEENL</span><br><span class="line">LSLKNSLNDCSNQVILAKASQEHHLSEETKCSASLFSSQCSELEDLTANTNTQDPFLIGS</span><br><span class="line">SKQMRHQSESQGVGLSDKELVSDDEERGTGLEENNQEEQSMDSNLGEAASGCESETSVSE</span><br><span class="line">DCSGLSSQSDILTTQQRDTMQHNLIKLQQEMAELEAVLEQHGSQPSNSYPSIISDSSALE</span><br><span class="line">DLRNPEQSTSEKAVLTSQKSSEYPISQNPEGLSADKFEVSADSSTSKNKEPGVERSSPSK</span><br><span class="line">CPSLDDRWYMHSCSGSLQNRNYPSQEELIKVVDVEEQQLEESGPHDLTETSYLPRQDLEG</span><br><span class="line">TPYLESGISLFSDDPESDPSEDRAPESARVGNIPSSTSALKVPQLKVAESAQSPAAAHTT</span><br><span class="line">DTAGYNAMEESVSREKPELTASTERVNKRMSMVVSGLTPEEFMLVYKFARKHHITLTNLI</span><br><span class="line">TEETTHVVMKTDAEFVCERTLKYFLGIAGGKWVVSYFWVTQSIKERKMLNEHDFEVRGDV</span><br><span class="line">VNGRNHQGPKRARESQDRKIFRGLEICCYGPFTNMPTDQLEWMVQLCGASVVKELSSFTL</span><br><span class="line">GTGVHPIVVVQPDAWTEDNGFHAIGQMCEAPVVTREWVLDSVALYQCQELDTYLIPQIPH</span><br><span class="line">SHY</span><br><span class="line"></span><br><span class="line">&gt;21A [CPC](http://cpc.cbi.pku.edu.cn/)</span><br><span class="line">AAATAGTTGACCAAGTGTGGTGGCTCACGTAGTCCCAGCACTTTGGGAGGCTGAGGCAGGAGGATCACTTGAGCCCAGGAATTTGAGACCAGCTTGGGCAACATAGTGAGACCTCATCTCTTAAAAAAAAAAATTAGCTGGGTGTGGTAGTGCACACCTGTGGTCCCAGCTACTTTAGAGGCTGAGGTAGAGGATTGCTTGAGCCTGGGAAGTTGGGGCTGTAGTGAGCTTTGATTGCATCACTGCACTCCAGCCTGGGTGACAGAGCAAGACCCTGTCTCTAAAAAATTAAATAAATAATAAAAAAATTAAAAAGTAACTCCC</span><br></pre></td></tr></table></figure>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/certificate.png" alt="certificate"></p>
<p><strong>从数学物理，到化学材料的科学理论突破，应用到工程界，引发信息编程等领域链式的技术革命，极大地推动了生物、医学等下游学科的发展。随着实验检测技术和试验设计的革新，人群研究逐渐能够摆脱伦理学的困境，内外暴露物质的测量方法也开始了急速地发展，直接从人类身上全面地获取有循证意义的数据成为可能。经典流行病学也将逆转被动研究黑箱问题的局面，使用生物信息学等方法，借助云储存、云计算分析海量组学数据，定量地研究全时空下暴露、分子、结局全水平的互作（interaction），彻底探清疾病机制，构建在人类医学生具有划时代意义的系统流行病学。而本就是限于伦理，“曲线救国”来研究人类疾病的动物实验，其现实意义也将大打折扣，甚至会在不久的将来只有博物馆的一席之地。</strong></p>
<blockquote>
<p>“Genetics Loads the Gun, Lifestyle Pulls the Trigger.”<br>—— Barbara O’Neill</p>
</blockquote>
<p><img src="http://care.diabetesjournals.org/content/diacare/36/5/1413/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" alt="Image of Interaction Network"><br>(Source: Franks PW, et al. Diabetes Care, 2013)</p>
<blockquote>
<p>从暴露组学发展到打开流行病学“黑箱子”的遗传组学、代谢组学等，系统流行病学可以多水平、多层次地考虑疾病的发生机制；除了机理研究之外，健康的社会决定因素和遗传易感性的影响因素研究，都进一步拓展了病因学研究的广度和深度。<br>——<a href="http://sph.pku.edu.cn/content/?2249.html" target="_blank" rel="noopener">李立明</a></p>
</blockquote>
<h2 id="华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师"><a href="#华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师" class="headerlink" title="华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师"></a>华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师</h2><ul>
<li><a href="http://gwxy.tjmu.edu.cn/info/1087/1796.htm" target="_blank" rel="noopener">龚静 博士</a></li>
<li><a href="http://chaolongwang.github.io" target="_blank" rel="noopener">王超龙 教授</a></li>
</ul>
<h2 id="参考书目"><a href="#参考书目" class="headerlink" title="参考书目"></a>参考书目</h2><ol>
<li>《生物信息学》, 李霞, 雷健波; 人民卫生出版社, 2015</li>
<li>《概率论与数理统计》, 盛骤, 谢式千, 潘承毅; 高等教育出版社, 2008</li>
</ol>
<h2 id="编程语言"><a href="#编程语言" class="headerlink" title="编程语言"></a>编程语言</h2><ul>
<li>Linux Shell / Bash<ul>
<li><a href="http://linux.vbird.org/linux_basic/" target="_blank" rel="noopener">鳥哥的Linux私房菜-基礎學習篇</a></li>
<li><a href="https://legacy.gitbook.com/book/wizardforcel/vbird-linux-basic-4e/details" target="_blank" rel="noopener">鸟哥的Linux私房菜：基础学习篇 第四版</a></li>
</ul>
</li>
<li>Git<ul>
<li><a href="http://iissnan.com/progit/" target="_blank" rel="noopener">《Pro Git》</a></li>
</ul>
</li>
<li>R<ul>
<li>《R语言实战》</li>
<li>《R语言与Bioconductor生物信息学应用》</li>
</ul>
</li>
<li>Perl<ul>
<li>《Perl语言入门》/《Learning Perl》传说中的“小骆驼”</li>
</ul>
</li>
<li>Python</li>
<li>C</li>
</ul>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/lncRNA/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/lncRNA/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/transcriptome/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/transcriptome/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><p><a href="https://mp.weixin.qq.com/s?__biz=MzAxMDkxODM1Ng==&amp;mid=2247484843&amp;idx=1&amp;sn=95370393e9d16e09e854adce78c94523&amp;chksm=9b484510ac3fcc064b1bfc19f5cc73a4fbc113a840a40c3a95632127c54f570d9d66550ebc22&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">任务清单</a>  <a href="https://mp.weixin.qq.com/s?__biz=MzAxMDkxODM1Ng==&amp;mid=2247485440&amp;idx=1&amp;sn=392c8649e334af99ac60e903aaec01e9&amp;chksm=9b4848bbac3fc1ad02ac2f11fd9f4440fb13cffc063d19da56fe34317d0621a4972ec26a41d1&amp;mpshare=1&amp;scene=23&amp;srcid=04136Qg8PRlWmG5bwOcYtjwq%23rd" target="_blank" rel="noopener">参考流程</a></p>
<h3 id="计算机资源的准备"><a href="#计算机资源的准备" class="headerlink" title="计算机资源的准备"></a>计算机资源的准备</h3><ul>
<li><p>系统环境及硬件配置：</p>
<ul>
<li>macOS High Sierra 10.13.4</li>
<li>MacBool Air (13-inch, Early 2015)</li>
<li>处理器   1.6GHz intel Core i5</li>
<li>内存  4GB 1600MHz DDR3</li>
<li>存储  128G</li>
<li><code>uname -a</code>  x86_64</li>
</ul>
</li>
<li><p>安装Homebrew包管理，要求预先安装Xcode；<br>（App Store中Xcode要求macOS High Sierra 10.13.2以上）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"</span><br></pre></td></tr></table></figure>
</li>
<li><p>用homebrew安装wget；（也可在<a href="rudix.org/packages/wget.html">rudix网站</a>上下载wget安装包）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">brew install wget</span><br></pre></td></tr></table></figure>
</li>
<li><p><a href="https://zhuanlan.zhihu.com/p/25085567" target="_blank" rel="noopener">wget下载miniconda</a>；<br>此处注意网上教程多在Linux或Windows虚拟Ubantu系统下操作，要注意下载MacOSX版文件！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh</span><br></pre></td></tr></table></figure>
</li>
<li><p>bash安装miniconda；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">bash Miniconda3-latest-MacOSX-x86_64.sh</span><br></pre></td></tr></table></figure>
</li>
<li><p>输入yes接受条款；</p>
</li>
<li><p>回车确认安装位置；</p>
</li>
<li><p>输入yes确定PATH；</p>
</li>
<li><p>source完成最后安装；<br>此处注意MacOSX系统中不是<code>.bashrc</code>，应根据反馈结果修改为<code>.bash_profile</code>！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">source ~/.bash_profile</span><br></pre></td></tr></table></figure>
</li>
<li><p>查看是否安装成功；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda</span><br></pre></td></tr></table></figure>
</li>
<li><p>更新miniconda；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda update conda</span><br></pre></td></tr></table></figure>
</li>
<li><p>添加channels,即为conda增加软件源；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">conda config - add channels r</span><br><span class="line">conda config - add channels bioconda</span><br><span class="line">conda config - add channels defaults</span><br><span class="line">conda config – add channels conda-forge</span><br></pre></td></tr></table></figure>
</li>
<li><p>查看添加的频道；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda config – get channels</span><br></pre></td></tr></table></figure>
</li>
<li><p>添加清华源镜像；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">conda config - add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/</span><br><span class="line">conda config - add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/</span><br><span class="line">conda config - add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/msys2/</span><br></pre></td></tr></table></figure>
</li>
<li><p>显示已添加的源镜像地址；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda config --set show_channel_urls yes</span><br></pre></td></tr></table></figure>
</li>
<li><p>安装sratoolkit,fastqc,hisats,samtools,htseq-count,R,Rstudio软件；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">conda install -c bioconda sra-tools</span><br><span class="line">conda install -c bioconda fastqc</span><br><span class="line">conda install -c bioconda hisat2</span><br><span class="line">conda install -c bioconda samtools</span><br><span class="line">conda install -c bioconda htseq</span><br><span class="line"><span class="meta">#</span> 本机已安装过R和RStudio,故未重复下载</span><br><span class="line"><span class="meta">#</span> conda install r</span><br><span class="line"><span class="meta">#</span> conda install rstudio</span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="读文章拿到测序数据"><a href="#读文章拿到测序数据" class="headerlink" title="读文章拿到测序数据"></a>读文章拿到测序数据</h3><ul>
<li><p>检索实战学习示例文章；<br><a href="https://www.nature.com/articles/ncomms13347" target="_blank" rel="noopener">AKAP95 regulates splicing through scaffolding RNAs and RNA processing factors. Nat Commun 2016 Nov 8;7:13347. PMID: 27824034</a></p>
</li>
<li><p>阅读文章Methods部分，尤Bioinformatic analyses和Data availability部分；</p>
<blockquote>
<p>Bioinformatic analyses<br>We obtained 35–55 million of paired 51 bp reads for each sample. The investigator was blinded to sample identities when performing bioinformatic analyses. All the reads were mapped to the <strong>human reference genome (GRCh37/hg19)</strong> using <strong>TopHat (v2.0.13)</strong> with a <strong>gene transfer file (GTF version GRCh37.70)</strong>. <strong>Low quality mapped reads (MQ&gt;30)</strong> were removed from the analysis. The <strong>mean insert sizes and the s.d.’s</strong> were calculated using <a href="http://broadinstitute.github.io/picard/" target="_blank" rel="noopener"><strong>Picard-tools (v1.126)</strong></a>.<br>For mRNA-Seq, <strong>read count tables</strong> were generated using <strong>HTSeq (v0.6.0)53</strong> and <strong>deferential expression (DE) analysis of genes</strong> were performed using <strong>DESeq (v3.0)54</strong>. <strong>Deferential exon usage</strong> was performed using <strong>DEXSeq (v3.1)29</strong>. Two independent biological repeats for control and KD were included in both DEseq and DEXseq. <em><strong>For 293 cells, the mRNA-seq results of the control samples include (1) those from the doxycycline-treated parental Flp-In T-REx 293 cells by us and (2) those from the doxycycline-treated control Flp-In T-REx 293 cells performed by another group unrelated to us (sample GSM1095127 in GSE44976)31, and the two AKAP95 KD samples were from the two different clones (miR#8 and miR#12) of Flp-In T-REx 293 cells stably expressing AKAP95 microRNAs after doxycycline treatment.</strong></em> For mouse ES cells, the two biological repeats for both control and shRNA#1-mediated Akap95 KD were independently generated by two different persons in our lab. The <strong>read per million normalized BigWig files</strong> were generated using <strong>BEDTools (v2.17.0)55</strong> and <strong>bedGraphToBigWig tool (v4)</strong>. All the <strong>downstream statistical analyses</strong> and generating <strong>plots</strong> were performed in <a href="http://www.r-project.org/" target="_blank" rel="noopener"><strong>R (v3.1.1)</strong></a>. <strong>Gene ontology analysis</strong> was performed at <a href="http://david.ncifcrf.gov/" target="_blank" rel="noopener"><strong>DAVID</strong></a>.<br>For RIP-Seq, mapped files were passed to Model-based Analysis of ChIP-Seq (MACS) (v1.4.2 20120305)56 for peak calling and <a href="http://homer.salk.edu/homer/" target="_blank" rel="noopener">HOMER (vv3.12, 6-8-2012)</a>57 was used for Motif finding. Bedgraph files generated by MACS were then normalized based on read per million and converted to BigWig files using bedGraphToBigWig. The input RNA-seq files were not subject to MACS. The genomic regions for annotation (intergenic regions, exons, UTRs, distal and proximal introns) were extracted from the GTF file used above. Peak annotations and coverage calculations for these genomic regions were performed using Bedtools and SAMtools (v0.1.18)58, respectively. Profile plots were generated by ngs.plot (v2.47)59.</p>
</blockquote>
</li>
</ul>
<blockquote>
<p>Data availability<br>The RIP-seq an RNA-seq data have been deposited in the Gene Expression Omnibus database, with accession code <strong>GSE81916</strong>. All other data is available from the author upon reasonable request.</p>
</blockquote>
<ul>
<li>在<a href="https://www.ncbi.nlm.nih.gov/geo/" target="_blank" rel="noopener">GEO</a>上查找<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81916" target="_blank" rel="noopener"><strong>GSE81916</strong></a>,查找得其对应的SRA号为<strong>SRP075747</strong>（以及BioProject号<strong>PRJNA323422</strong>）；</li>
</ul>
<blockquote>
<p>Overall design<br>Samples 1-8 are RNA-immunoprecipitation (RIP)-seq to determine AKAP95 binding to the transcriptome. <strong>Samples 9-15 are mRNA-seq</strong> to determine effect of AKAP95 knockdown in human 293 cells (9-11) or mouse ES cells (12-15).</p>
</blockquote>
<ul>
<li><p>阅读网页Overall design部分介绍得知，属于RNA-seq数据的是第9-15个样品；</p>
</li>
<li><p>在<a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=search_obj" target="_blank" rel="noopener">SRA</a>上查找<a href="https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP075747&amp;go=go" target="_blank" rel="noopener"><strong>SRP075747</strong></a>（或<a href="https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA323422" target="_blank" rel="noopener"><strong>PRJNA323422</strong></a>）,查找得其第9-15个样品，即<strong>GSM2177723</strong>到<strong>GSM2177729</strong>对应的SRA号为<strong>SRR3589956</strong>到<strong>SRR3589962</strong>；<br><em>注意理解GEO/SRA数据库的数据存放形式</em></p>
</li>
<li><p>安装axel来下载数据比wget及prefetch更快；（也可使用<a href="http://www.bio-info-trainee.com/bioconductor_China/software/GEOquery.html" target="_blank" rel="noopener">GEOquery</a>下载）<br>（Aspera只有Windows系统下的client）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">brew install axel</span><br></pre></td></tr></table></figure>
</li>
<li><p>循环axel语句实现数据下载；<br>（<strong>2200+KB/s，用时约两小时</strong>）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">for i in &#123;56..62&#125;  # &#123;56..62&#125;与`seq 57 62`等价</span><br><span class="line">do</span><br><span class="line">  axel ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/SRP075/SRP075747/SRR35899$i/SRR35899$i.sra</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="了解fastq测序数据"><a href="#了解fastq测序数据" class="headerlink" title="了解fastq测序数据"></a>了解fastq测序数据</h3><ul>
<li><p>循环sratoolkit（sra-tools）中fastq-dump语句把sra文件转换为fastq格式；<br>（<strong>用时约三个半小时</strong>）<br>注意这一步不显示百分数进度条！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">fastq-dump -h # 查看帮助文件</span><br><span class="line">for i in &#123;56..62&#125;</span><br><span class="line">do</span><br><span class="line">  fastq-dump --gzip --split-3 -O /Users/xiapengfei/ -A SRR35899$&#123;i&#125;.sra</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
<li><p>循环fastqc语句测试测序文件的质量；(<a href="http://fbb84b26.wiz03.com/share/s/3XK4IC0cm4CL22pU-r1HPcQQ1iRTvV2GwkwL2AaxYi2fXHP7" target="_blank" rel="noopener">MultiQC可以批量显示质控结果</a>）<br>（<strong>4线程，用时约25分钟</strong>）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">for i in &#123;56..62&#125;</span><br><span class="line">do</span><br><span class="line">  fastqc -t 4 -O /Users/xiapengfei/ SRR35899$i.sra_1.fastq.gz SRR35899$i.sra_2.fastq.gz</span><br><span class="line">done  # fastqc *.fastq在当前目录中无其他同类文件时等价，其他循环或可同理</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p>_<a href="https://github.com/XIAPengfei1996/Shia/tree/master/bioinfo/fastqc_html" target="_blank" rel="noopener">理解测序reads，GC含量，质量值，接头，index，fastqc的全部报告</a>_</p>
<p><a href="http://www.bio-info-trainee.com/2275.html" target="_blank" rel="noopener"><strong>经验教训</strong></a></p>
<h3 id="了解参考基因组及基因注释"><a href="#了解参考基因组及基因注释" class="headerlink" title="了解参考基因组及基因注释"></a>了解参考基因组及基因注释</h3><ul>
<li>在<a href="http://genome.ucsc.edu/index.html" target="_blank" rel="noopener">UCSC</a>中打开Downloads中Genome Data页面，点击Human选择Feb. 2009 (GRCh37/hg19)栏下Full data set，下拉找到chromFa.tar.gz文件；</li>
</ul>
<blockquote>
<p>chromFa.tar.gz - The assembly sequence in one file per chromosome.<br>    Repeats from RepeatMasker and Tandem Repeats Finder (with period<br>    of 12 or less) are shown in lower case; non-repeating sequence is<br>    shown in upper case.</p>
</blockquote>
<ul>
<li><p>axel命令下载hg19参考基因组数据；<br>（<strong>400+KB/s，用时约35分钟</strong>）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">axel http://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/chromFa.tar.gz</span><br><span class="line">tar zxvf chromFa.tar.gz  # 解压该文件</span><br><span class="line">cat *.fa &gt; hg19.fa  # 将解压文件合并为一个文件</span><br><span class="line">rm chr*.fa  # 删除无关数据</span><br></pre></td></tr></table></figure>
</li>
<li><p>从<a href="http://www.gencodegenes.org" target="_blank" rel="noopener">gencode数据库</a>中打开Data中Human栏内<a href="https://www.gencodegenes.org/releases/grch37_mapped_releases.html" target="_blank" rel="noopener">GRCh37-mapped Releases</a>，点击GENCODE release栏下最新版的数据，下载基因注释文件；<br>（<strong>用时5分钟</strong>)</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">axel ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/GRCh37_mapping/gencode.v28lift37.annotation.gtf.gz  # GTF格式主要是用来描述基因的注释</span><br><span class="line">axel ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/GRCh37_mapping/gencode.v28lift37.annotation.gff3.gz  # GFF文件是一种用来描述基因组特征的文件，现在我们所使用的大部分都是第三版（GFF3）</span><br><span class="line"><span class="meta">#</span> gzip -d解压并删除压缩包</span><br><span class="line">gzip -d gencode.v28lift37.annotation.gtf.gz</span><br><span class="line">gzip -d gencode.v28lift37.annotation.gff3.gz</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>- 用IGV去查看感兴趣的基因的结构；</em></p>
<ul>
<li>下载安装<a href="https://java.com/en/download/mac_download.jsp" target="_blank" rel="noopener">JAVA 8</a>；</li>
<li>下载<a href="http://software.broadinstitute.org/software/igv/download" target="_blank" rel="noopener">igv</a>直接打开；</li>
<li>在菜单栏Genomes中选择Load Genome from File…导入hg19.fa文件；</li>
<li>在菜单栏Tools中打开Run igvtools…界面，在Command栏中选择Sort，在Input File栏中导入gtf文件后运行；</li>
<li>在菜单栏File中选择Load from File…导入sorted.gtf文件并index；</li>
<li>在工具栏输入查找的基因名词（如：TP53）或染色体定位（如chr17:7,569,720-7,592,868)查看<a href="https://github.com/XIAPengfei1996/Shia/tree/master/bioinfo/igv" target="_blank" rel="noopener">可视化结构</a>；</li>
</ul>
<h3 id="序列比对"><a href="#序列比对" class="headerlink" title="序列比对"></a>序列比对</h3><ul>
<li>从<a href="http://ccb.jhu.edu/software/hisat2/index.shtml" target="_blank" rel="noopener">HISAT2</a>的主页下载index文件；<br>（<strong>400+KB/s，用时约两个半+小时</strong>）<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">axel ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/data/hg19.tar.gz  # 下载人类的索引</span><br><span class="line">axel ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/data/mm10.tar.gz  # 下载小鼠的索引</span><br><span class="line">tar -zxvf *.tar.gz  # 解压压缩包</span><br><span class="line">rm -rf *.tar.gz  # 删除压缩包</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p>这次分析是人类mRNA-Seq测序的结果，但是这里其实只下载了3个人类的sra文件。一般而言RNA-Seq数据分析都至少要有2个重复，所以必须要有4个sra文件才行。实际上前面原文Bioinformatic analyses部分有说该研究control组中的一组数据用的是其他课题组的：</p>
<blockquote>
<p><em><strong>For 293 cells, the mRNA-seq results of the control samples include (1) those from the doxycycline-treated parental Flp-In T-REx 293 cells by us and (2) those from the doxycycline-treated control Flp-In T-REx 293 cells performed by another group unrelated to us (sample GSM1095127 in GSE44976)31, and the two AKAP95 KD samples were from the two different clones (miR#8 and miR#12) of Flp-In T-REx 293 cells stably expressing AKAP95 microRNAs after doxycycline treatment.</strong></em></p>
</blockquote>
<p>不同来源的RNA-Seq数据的构建文库不一样,结果之间比较需要考虑<strong>批次效应（batch effect）</strong> 的影响<br>IVT-seq reveals extreme bias in RNA-sequencing 证明了不同文库的RNA-Seq结果会存在很大差异</p>
<ul>
<li><p>把fastq格式的reads比对上去得到sam文件(仅小鼠，下同)；<br>注意这一步也不显示百分数进度条！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br></pre></td><td class="code"><pre><span class="line"><span class="meta">#</span> 比对人类的索引 9:55</span><br><span class="line"><span class="meta">#</span> for i in &#123;56..58&#125;</span><br><span class="line"><span class="meta">#</span> do</span><br><span class="line"><span class="meta">#</span>   hisat2 -t -x /Users/xiapengfei/hg19/genome -1 SRR35899$&#123;i&#125;.sra_1.fastq.gz -2 SRR35899$&#123;i&#125;.sra_2.fastq.gz -S SRR35899$&#123;i&#125;.sam</span><br><span class="line"><span class="meta">#</span> done</span><br><span class="line"></span><br><span class="line"><span class="meta">#</span> 比对小鼠的索引</span><br><span class="line">for i in &#123;59..62&#125;</span><br><span class="line">do</span><br><span class="line">  hisat2 -t -x /Users/xiapengfei/hg19/genome -1 SRR35899$&#123;i&#125;.sra_1.fastq.gz -2 SRR35899$&#123;i&#125;.sra_2.fastq.gz -S SRR35899$&#123;i&#125;.sam</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
<li><p>用samtools把sam文件转为bam文件，并且排序(注意N和P两种排序区别)索引好；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line">for i in &#123;59..62&#125;</span><br><span class="line">do </span><br><span class="line">  samtools view -S SRR35899$&#123;i&#125;.sam -b &gt; SRR35899$&#123;i&#125;.bam  # 将比对后的sam格式文件转换成bam文件</span><br><span class="line">  samtools sort SRR35899$&#123;i&#125;.bam -o SRR35899$&#123;i&#125;.sorted.bam  # 将所有的bam文件进行排序</span><br><span class="line">  samtools index SRR35899$&#123;i&#125;_sorted.bam # 将所有的排序文件建立索引</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>samtools sort默认按照染色体位置进行排序，-n参数根据read名进行排序。-t参数根据TAG进行排序</em></p>
<ul>
<li>对bam文件进行质控；<br>可以使用<a href="http://rseqc.sourceforge.net/" target="_blank" rel="noopener">RSeQC</a><br><em>RSeQC的安装，需要预先安装R，Python2.7，gcc，<a href="http://www.jianshu.com/p/14fd4de54402" target="_blank" rel="noopener">numpy</a></em><figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">pip install RSeQC</span><br><span class="line"><span class="keyword">for</span> i <span class="keyword">in</span> &#123;<span class="number">56.</span><span class="number">.62</span>&#125;</span><br><span class="line">do</span><br><span class="line">  bam_stat.py -i SRR35899$&#123;i&#125;_sorted.bam</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>另有<a href="http://broadinstitute.github.io/picard/" target="_blank" rel="noopener">Picard</a></em><br><figure class="highlight java"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">conda install ucsc-gtftogenepred</span><br><span class="line">gtfToGenePred -genePredExt -ignoreGroupsWithoutExons -geneNameAsName2 gencode.v28lift37.annotation.gtf gencode.v28lift37.annotation.gpd</span><br><span class="line">axel http:<span class="comment">//hgdownload.soe.ucsc.edu/goldenPath/hg19/database/refFlat.txt.gz</span></span><br><span class="line">gzip -d refFlat.txt.gz</span><br><span class="line">java -jar ~/biosoft/picard_2.8.0/picard.jar CollectRnaSeqMetrics I= Homo_sapiens_Control_293_cell_sorted.bam O=Homo_sapiens_Control_293_cell_RNA_Metrics REF_FLAT= ~/rna_seq/data/reference/genome/hg19/refFlat.txt STRAND_SPECIFICITY=NONE</span><br></pre></td></tr></table></figure></p>
<p><em>或<a href="http://qualimap.bioinfo.cipf.es/" target="_blank" rel="noopener">Qualimap</a>等工具</em></p>
<ul>
<li>IGV载入参考序列，注释和BAM文件查看略；</li>
</ul>
<h3 id="reads计数"><a href="#reads计数" class="headerlink" title="reads计数"></a>reads计数</h3><p>用htseq对比得到的bam文件进行count计数；<br><figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">for i in $(seq 56 58)</span><br><span class="line">do </span><br><span class="line">  htseq-count -f bam -r pos -s no SRR35899$&#123;i&#125;.bam gencode.vM10.annotation.gtf SRR35899$&#123;i&#125;.count</span><br><span class="line">done</span><br></pre></td></tr></table></figure></p>
<ul>
<li>将count文件合并为表达矩阵；（59和61是control， 60和62是repeat）<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br></pre></td><td class="code"><pre><span class="line">options(stringsAsFactors = <span class="literal">FALSE</span>)</span><br><span class="line"><span class="comment"># 导入表格数据</span></span><br><span class="line">control1 &lt;- read.table(<span class="string">"SRR3589959.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"control"</span>))</span><br><span class="line">repeat1 &lt;- read.table(<span class="string">"SRR3589960.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"akap951"</span>))</span><br><span class="line">control1 &lt;- read.table(<span class="string">"SRR3589961.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"control"</span>))</span><br><span class="line">repeat2 &lt;- read.table(<span class="string">"SRR3589962.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"akap952"</span>))</span><br><span class="line"><span class="comment"># 按照gene_id合并</span></span><br><span class="line">raw_count &lt;- merge(merge(control1, control2, by=<span class="string">"gene_id"</span>), merge(repeat1, repeat2, by=<span class="string">"gene_id"</span>))</span><br><span class="line"><span class="comment"># 过滤删去5行</span></span><br><span class="line">raw_count_filt &lt;- raw_count[-<span class="number">48823</span>:-<span class="number">48825</span>, ]</span><br><span class="line">raw_count_filter &lt;- raw_count_filt[-<span class="number">1</span>:-<span class="number">2</span>, ]</span><br><span class="line"><span class="comment"># 因为EBI数据库无法直接搜索小数位，因此需要替换为整数形式</span></span><br><span class="line">ENSEMBL &lt;- gsub(<span class="string">"\\.\\d*"</span>, <span class="string">""</span>, raw_count_filter$gene_id)</span><br><span class="line"><span class="comment"># 将ENSEMBL重新添加到矩阵中</span></span><br><span class="line">row.names(raw_count_filter) &lt;- ENSEMBL</span><br><span class="line"><span class="comment"># 在EBI或UniProt数据库找到AKAP95的id，并从矩阵中找到访问，并赋值给AKAP95变量</span></span><br><span class="line">&gt; AKAP95 &lt;- raw_count_filt[rownames(raw_count_filt)==<span class="string">"ENSG00000105127"</span>,]</span><br><span class="line"><span class="comment"># 查看AKAP95基因表达情况，结果明显比对照组高</span></span><br><span class="line">&gt; AKAP95</span><br><span class="line"><span class="comment"># 查看总体情况</span></span><br><span class="line">summary(raw_count_filter)</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>另可使用perl</em><br><figure class="highlight perl"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br></pre></td><td class="code"><pre><span class="line">perl -lne <span class="string">'if($ARGV=~/Homo_sapiens_(.*)_sorted.count/)&#123;print "$1\t$_"&#125;'</span> *|<span class="keyword">grep</span> -v  Homo_sapiens&gt;hg.count</span><br><span class="line"><span class="comment"># 先把所有文件进行合并</span></span><br><span class="line">setwd(<span class="string">"~/rna_seq/work/matrix"</span>)</span><br><span class="line">hg &lt;- read.csv(file = <span class="string">"hg.count"</span>,header = F,sep = <span class="string">"\t"</span>)</span><br><span class="line">colnames(hg) &lt;- c(<span class="string">'sample'</span>,<span class="string">'gene'</span>,<span class="string">'count'</span>)</span><br><span class="line">library(reshape2)</span><br><span class="line">reads &lt;- dcast(hg,formula = gene ~ sample)</span><br><span class="line">write.table(reads,file = <span class="string">"hg_join.count"</span>,sep = <span class="string">"\t"</span>,quote = FALSE,row.names = FALSE)</span><br><span class="line">summary(reads)</span><br><span class="line">sample_mean &lt;- group_by(hg,sample)%&gt;% summarize(mean=mean(count))</span><br><span class="line">gene_mean &lt;- group_by(hg,gene)%&gt;% summarize(mean=mean(count)) </span><br><span class="line">tmp &lt;- full_join(hg,gene_mean,by=<span class="string">'gene'</span>)</span><br></pre></td></tr></table></figure></p>
<h3 id="差异基因分析"><a href="#差异基因分析" class="headerlink" title="差异基因分析"></a>差异基因分析</h3><p>参考：<br><a href="https://zhuanlan.zhihu.com/p/30350531" target="_blank" rel="noopener">青山屋主</a>  <a href="https://mp.weixin.qq.com/s/1mda7NpOxShbOxZKSJar7Q" target="_blank" rel="noopener">徐洲更</a>  <a href="https://mp.weixin.qq.com/s?__biz=MzIwNTEwMTUyOQ==&amp;mid=2649694709&amp;idx=1&amp;sn=40b354fef74d3adf5aac7b8745303af2&amp;chksm=8f2d8143b85a0855404621656c5946acc4919cc61a65719c450a461ecbdeb1ff56c72fe798bb&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">PANDA姐</a>/<a href="https://mp.weixin.qq.com/s?__biz=MzIwNTEwMTUyOQ==&amp;mid=2649694735&amp;idx=1&amp;sn=5ad87d24643c08ab945288e086da4359&amp;chksm=8f2d82b9b85a0baf92cc6af6f647c7e89710138a3f95e5e0a939894f21034edc57d5f7a30f52&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">沈梦圆</a></p>
<ul>
<li><p>在R中安装DESeq2；（也可使用edgeR，limma等）</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">rm(list=ls())</span><br><span class="line"><span class="keyword">source</span>(<span class="string">"https://bioconductor.org/biocLite.R"</span>) <span class="comment"># 载入安装工具</span></span><br><span class="line">biocLite(<span class="string">"DESeq2"</span>)  <span class="comment"># 安装包</span></span><br><span class="line"><span class="keyword">library</span>(<span class="string">"DESeq2"</span>) <span class="comment"># 加载包，测试是否安装成功</span></span><br></pre></td></tr></table></figure>
</li>
<li><p>正式构建dds矩阵；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br></pre></td><td class="code"><pre><span class="line">countData &lt;- raw_count_filter</span><br><span class="line">condition &lt;- factor(c(<span class="string">"control"</span>,<span class="string">"control"</span>,<span class="string">"KD"</span>,<span class="string">"KD"</span>))</span><br><span class="line">colData &lt;- data.frame(row.names=colnames(countData), condition)</span><br><span class="line">dds &lt;- DESeqDataSetFromMatrix(countData, DataFrame(condition), design= ~ condition )</span><br><span class="line">head(dds)  <span class="comment">#查看一下构建好的矩阵</span></span><br><span class="line">dds &lt;- DESeq(dds)   <span class="comment">#对原始dds进行normalize</span></span><br><span class="line"><span class="comment"># 查看结果的名称，本次实验中是 "Intercept"，"condition_akap95_vs_control"</span></span><br><span class="line">resultsNames(dds2)</span><br><span class="line"><span class="comment"># 将结果用results()函数来获取，赋值给res变量</span></span><br><span class="line">res &lt;- results(dds2)</span><br><span class="line"><span class="comment"># summary一下，看一下结果的概要信息</span></span><br><span class="line">summary(res)</span><br></pre></td></tr></table></figure>
</li>
<li><p>提取差异分析结果；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># 获取padj（p值经过多重校验校正后的值）小于0.05，表达倍数取以2为对数后大于1或者小于-1的差异表达基因。</span></span><br><span class="line">&gt; table(res$padj&lt;<span class="number">0.05</span>) <span class="comment">#取P值小于0.05的结果</span></span><br><span class="line">&gt; res &lt;- res[order(res$padj),]</span><br><span class="line">&gt; diff_gene_deseq2 &lt;-subset(res,padj &lt; <span class="number">0.05</span> &amp; (log2FoldChange &gt; <span class="number">1</span> | log2FoldChange &lt; -<span class="number">1</span>))</span><br><span class="line">&gt; diff_gene_deseq2 &lt;- row.names(diff_gene_deseq2)</span><br><span class="line">&gt; resdata &lt;-  merge(as.data.frame(res),as.data.frame(counts(dds2,normalize=<span class="literal">TRUE</span>)),by=<span class="string">"row.names"</span>,sort=<span class="literal">FALSE</span>)</span><br><span class="line"><span class="comment"># 得到csv格式的差异表达分析结果</span></span><br><span class="line">&gt; write.csv(resdata,file= <span class="string">"control_vs_akap95.cvs"</span>,row.names = <span class="literal">F</span>)</span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="差异基因结果注释-富集分析"><a href="#差异基因结果注释-富集分析" class="headerlink" title="差异基因结果注释/富集分析"></a>差异基因结果注释/富集分析</h3><p>参考：<br><a href="https://mp.weixin.qq.com/s/p-n5jq5Rx2TqDBStS2nzoQ" target="_blank" rel="noopener">徐洲更</a>  <a href="https://www.jianshu.com/p/95c0472f5a8e" target="_blank" rel="noopener">宇天</a>  <a href="https://mp.weixin.qq.com/s?__biz=MzIwNTEwMTUyOQ==&amp;mid=2649694917&amp;idx=1&amp;sn=a318f8cf98f306d46963986011c73600&amp;chksm=8f2d8273b85a0b65270a986f7bb28e8efa7fd15309505a5a026bbedaafaff7e30e4d4f13dd02&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">沈梦圆</a></p>
<ul>
<li><p>根据padj &lt; 0.05 且Log2FoldChange的绝对值大于1的标准筛选显著差异表达基因集；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">deseq2.sig &lt;- subset(res, padj &lt; <span class="number">0.05</span> &amp; abs(log2FoldChange) &gt; <span class="number">1</span>)</span><br></pre></td></tr></table></figure>
</li>
<li><p>安装下载注释数据包；（rog.HS.eg.db)</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">source</span>(<span class="string">"https://bioconductor.org/biocLite.R"</span>)</span><br><span class="line">biocLite(<span class="string">"clusterProfiler"</span>)</span><br><span class="line">biocLite（<span class="string">"AnnotationHub"</span>)</span><br><span class="line"><span class="keyword">library</span>(AnnotationHub)</span><br><span class="line">ah &lt;- AnnotationHub()</span><br><span class="line">org.hs &lt;- ah[[<span class="string">'AH53766'</span>]]</span><br></pre></td></tr></table></figure>
</li>
<li><p>GO富集和GESA；<br><em>Y叔的clusterProfiler简单好用，结果正确，没有必要专门导出数据用GSEA软件做超几何分布检验分析了</em></p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br></pre></td><td class="code"><pre><span class="line">ego &lt;- enrichGO(</span><br><span class="line">  gene = row.names(deseq2.sig),</span><br><span class="line">  OrgDb = org.hs,</span><br><span class="line">  keytype = <span class="string">"ENSEMBL"</span>,</span><br><span class="line">  ont = <span class="string">"MF"</span></span><br><span class="line">)</span><br><span class="line">dotplot(ego,font.size=<span class="number">5</span>)  <span class="comment"># 气泡图</span></span><br><span class="line">enrichMap(ego, vertex.label.cex=<span class="number">1.2</span>, layout=igraph::layout.kamada.kawai)</span><br><span class="line">plotGOgraph(ego)  <span class="comment"># 网络图</span></span><br><span class="line">gseaplot(gsemf, geneSetID=<span class="string">"GO:0004871"</span>)  <span class="comment"># 画GSEA标志性图</span></span><br></pre></td></tr></table></figure>
</li>
<li><p>KEGG富集分析；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">library</span>(clusterProfiler)</span><br><span class="line">gene_list &lt;- mapIds(org.hs, keys = row.names(deseq2.sig),</span><br><span class="line">  column = <span class="string">"ENTREZID"</span>, keytype = <span class="string">"ENSEMBL"</span> )</span><br><span class="line">kk &lt;- enrichKEGG(gene_list, organism=<span class="string">"hsa"</span>,</span><br><span class="line">  keyType = <span class="string">"ncbi-geneid"</span>,</span><br><span class="line">  pvalueCutoff=<span class="number">0.05</span>, pAdjustMethod=<span class="string">"BH"</span>,</span><br><span class="line">  qvalueCutoff=<span class="number">0.1</span>)</span><br><span class="line">head(summary(kk))</span><br><span class="line">dotplot(kk)  <span class="comment"># 气泡图</span></span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="致谢"><a href="#致谢" class="headerlink" title="致谢"></a>致谢</h3><p>感谢曾健明在其建立的生信技能树平台发起转录组入门实战项目；<br>感谢徐洲更、沈梦圆、青山屋主等在各平台上热心分享经验以资参考；<br>感谢所有论文作者和数据共享平台开放数据获取途径；<br>获益良多，相见恨晚！</p>
<h3 id="参考文献"><a href="#参考文献" class="headerlink" title="参考文献"></a>参考文献</h3><p>《RNA-seq数据分析实用方法》<br>（《RNA-seq Data Analysis A Practical Approach》）</p>
<h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
  </section>

  


          </div>
          


          

        </div>
        
          
  
  <div class="sidebar-toggle">
    <div class="sidebar-toggle-line-wrap">
      <span class="sidebar-toggle-line sidebar-toggle-line-first"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-middle"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-last"></span>
    </div>
  </div>

  <aside id="sidebar" class="sidebar">
    
    <div class="sidebar-inner">

      

      

      <section class="site-overview-wrap sidebar-panel sidebar-panel-active">
        <div class="site-overview">
          <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
            
              <p class="site-author-name" itemprop="name">John Doe</p>
              <p class="site-description motion-element" itemprop="description"></p>
          </div>

          <nav class="site-state motion-element">

            
              <div class="site-state-item site-state-posts">
              
                <a href="/archives/">
              
                  <span class="site-state-item-count">7</span>
                  <span class="site-state-item-name">posts</span>
                </a>
              </div>
            

            

            

          </nav>

          

          

          
          

          
          

          

        </div>
      </section>

      

      

    </div>
  </aside>


        
      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="footer-inner">
        <div class="copyright">&copy; <span itemprop="copyrightYear">2018</span>
  <span class="with-love">
    <i class="fa fa-user"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">John Doe</span>

  
</div>


  <div class="powered-by">Powered by <a class="theme-link" target="_blank" href="https://hexo.io">Hexo</a></div>



  <span class="post-meta-divider">|</span>



  <div class="theme-info">Theme &mdash; <a class="theme-link" target="_blank" href="https://github.com/iissnan/hexo-theme-next">NexT.Muse</a> v5.1.4</div>




        







        
      </div>
    </footer>

    
      <div class="back-to-top">
        <i class="fa fa-arrow-up"></i>
        
      </div>
    

    

  </div>

  

<script type="text/javascript">
  if (Object.prototype.toString.call(window.Promise) !== '[object Function]') {
    window.Promise = null;
  }
</script>









  












  
  
    <script type="text/javascript" src="/lib/jquery/index.js?v=2.1.3"></script>
  

  
  
    <script type="text/javascript" src="/lib/fastclick/lib/fastclick.min.js?v=1.0.6"></script>
  

  
  
    <script type="text/javascript" src="/lib/jquery_lazyload/jquery.lazyload.js?v=1.9.7"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/fancybox/source/jquery.fancybox.pack.js?v=2.1.5"></script>
  


  


  <script type="text/javascript" src="/js/src/utils.js?v=5.1.4"></script>

  <script type="text/javascript" src="/js/src/motion.js?v=5.1.4"></script>



  
  

  

  


  <script type="text/javascript" src="/js/src/bootstrap.js?v=5.1.4"></script>



  


  




	





  





  












  





  

  

  

  
  

  

  

  

</body>
</html>
