<!DOCTYPE html>



  


<html class="theme-next muse use-motion" lang="">
<head>
  <meta charset="UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=edge" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"/>
<meta name="theme-color" content="#222">









<meta http-equiv="Cache-Control" content="no-transform" />
<meta http-equiv="Cache-Control" content="no-siteapp" />
















  
  
  <link href="/lib/fancybox/source/jquery.fancybox.css?v=2.1.5" rel="stylesheet" type="text/css" />







<link href="/lib/font-awesome/css/font-awesome.min.css?v=4.6.2" rel="stylesheet" type="text/css" />

<link href="/css/main.css?v=5.1.4" rel="stylesheet" type="text/css" />


  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png?v=5.1.4">


  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png?v=5.1.4">


  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png?v=5.1.4">


  <link rel="mask-icon" href="/images/logo.svg?v=5.1.4" color="#222">





  <meta name="keywords" content="Hexo, NexT" />










<meta property="og:type" content="website">
<meta property="og:title" content="Hexo">
<meta property="og:url" content="http://yoursite.com/index.html">
<meta property="og:site_name" content="Hexo">
<meta property="og:locale" content="default">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Hexo">



<script type="text/javascript" id="hexo.configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Muse',
    version: '5.1.4',
    sidebar: {"position":"left","display":"post","offset":12,"b2t":false,"scrollpercent":false,"onmobile":false},
    fancybox: true,
    tabs: true,
    motion: {"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},
    duoshuo: {
      userId: '0',
      author: 'Author'
    },
    algolia: {
      applicationID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    }
  };
</script>



  <link rel="canonical" href="http://yoursite.com/"/>





  <title>Hexo</title>
  








</head>

<body itemscope itemtype="http://schema.org/WebPage" lang="default">

  
  
    
  

  <div class="container sidebar-position-left 
  page-home">
    <div class="headband"></div>

    <header id="header" class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-wrapper">
  <div class="site-meta ">
    

    <div class="custom-logo-site-title">
      <a href="/"  class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Hexo</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
      
        <p class="site-subtitle"></p>
      
  </div>

  <div class="site-nav-toggle">
    <button>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
      <span class="btn-bar"></span>
    </button>
  </div>
</div>

<nav class="site-nav">
  

  
    <ul id="menu" class="menu">
      
        
        <li class="menu-item menu-item-home">
          <a href="/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-home"></i> <br />
            
            Home
          </a>
        </li>
      
        
        <li class="menu-item menu-item-archives">
          <a href="/archives/" rel="section">
            
              <i class="menu-item-icon fa fa-fw fa-archive"></i> <br />
            
            Archives
          </a>
        </li>
      

      
    </ul>
  

  
</nav>



 </div>
    </header>

    <main id="main" class="main">
      <div class="main-inner">
        <div class="content-wrap">
          <div id="content" class="content">
            
  <section id="posts" class="posts-expand">
    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/11/25/nutrition&foodhygiene/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/11/25/nutrition&foodhygiene/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-11-25T20:00:15+08:00">
                2018-11-25
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="绪论"><a href="#绪论" class="headerlink" title="绪论"></a>绪论</h1><ul>
<li>营养与食品卫生学 nutrition &amp; food hygiene<br>预防医学领域内，主要研究膳食与机体的相互作用及其对健康的影响和作用机制，据此提出预防疾病和促进健康的措施、政策和法规等。</li>
</ul>
<ul>
<li>营养<br>机体从外界摄取食物，经过体内的消化、吸收和（或）代谢后，参与构建组织器官，满足生理功能或体力活动需要的必要的生物学过程。</li>
</ul>
<ul>
<li>营养学 nutrition science<br>研究食物中对人体有益的成分，以及人体摄取和利用这些成分以维持和促进健康的规律和机制，并在此基础上，采取具体的、宏观的、社会性措施，以改善人类健康，提高生命的质量的科学。</li>
</ul>
<ul>
<li>食品卫生学 food hygiene<br>研究食品中可能存在的、危害人体健康的有害因素，及其对机体的作用规律和机制，并在此基础上，提出具体、宏观的预防措施，以提高食品卫生质量，保护食用者安全的科学。</li>
</ul>
<h1 id="营养学"><a href="#营养学" class="headerlink" title="营养学"></a>营养学</h1><h2 id="基础营养学"><a href="#基础营养学" class="headerlink" title="基础营养学"></a>基础营养学</h2><ul>
<li>营养素（nutrient）：<br>为维持机体繁殖、生长发育和生存等一切生命活动和过程，需要从外界环境中摄取的物质。</li>
</ul>
<p>水的生理功能：</p>
<ol>
<li>构成细胞和体液的重要组成部分</li>
<li>参与新陈代谢</li>
<li>调节体温</li>
<li>润滑作用</li>
</ol>
<ul>
<li>植物化学物 phytochemicals<br>食物中除了含有多种营养素外，还含有其他许多对人体有益的物质，这些植物中食物的生物活性成分是植物能量代谢过程中产生的多种中间或末端低分子量次级代谢产物，具有保护人体、预防心血管病和癌症等慢性非传染性疾病的作用，现已统称为植物化学物。除个别是维生素的前提物质（如β-胡萝卜素）外，其余均为非传统营养素成分，故而过去称为非营养素生物活性成分。</li>
</ul>
<ul>
<li>合理营养（rational nutrition）：<br>是指人体每天从食物中摄入的能量和各种营养素的数量和相互之间的比例，能满足在不同生理阶段、不同劳动环境和不同劳动强度下的需要，并使机体处于良好的健康状态。</li>
</ul>
<ul>
<li>营养不良（malnutrition）：<br>是指由于一种或一种以上营养素的缺乏或过剩所造成的机体健康异常或疾病状态。<br>包括营养缺乏（nutrition deficiency）和营养过剩（nutrition excess）。</li>
</ul>
<ul>
<li>膳食营养素参考摄入量（dietary reference intakes, DRIs）：<br>是为了保证人体合理摄入营养素，避免缺乏和过量，再推荐膳食营养素供给量（recommended dietary allowance, RDA)的基础上发展起来的每日平均膳食营养素摄入量的一组参考值。</li>
</ul>
<ol>
<li>平均需要量（estimated average requirement, EAR）：<br>是指某一特定性别、年龄和生理状态群体中的所有个体对某种营养素需要量的平均值。<br>EAR是制定RNI的基础，可用于评价和计划群体的膳食摄入量，或判断个体某营养素摄入量不足的可能性。</li>
<li>推荐摄入量（RNI）：<br>是指可以满足某一特定性别、年龄和生理状态群体中绝大多数个体（97~98%）需要量的某种营养素摄入水平。<br>RNI可作为个体每日摄入某种营养素的推荐值/目标值。 # 能量需要量EER等于能力EAR。</li>
<li>适宜摄入量（adequate intake, AI）：<br>AI是通过观察或实验获得的健康人群某种营养素的摄入量。<br>AI与RNI一样可作为目标人群中个体每日摄入某种营养素的推荐值/目标值，但准确性不如RNI，可能偏高。</li>
<li>可耐受最高摄入量（tolerable upper intake level, UL）:<br>是营养素或事物成分每日摄入量的安全上限，即一个健康人群中几乎所有个体都不会产生毒副作用的最高摄入量。</li>
<li>宏量营养素可接受范围（acceptable macronutrient distribution ranges, AMDR）</li>
<li>预防慢性非传染性疾病的建议摄入量（proposed intakes for preventing non-communicable chronic diseases, PI-NCD）</li>
<li>特定建议值（specific proposed levels, SPL）</li>
</ol>
<ul>
<li>合理膳食（rational diet）：<br>又称平衡膳食（balanced diet），是指提供给机体种类齐全、数量充足、比例合适的能量和各种营养素，并与机体的需要保持平衡，进而达到合理营养、促进健康、预防疾病的膳食。是实现合理营养的物质基础和唯一途径。</li>
</ul>
<p>合理膳食要求：</p>
<ol>
<li>食物种类齐全，数量充足，比例合适</li>
<li>保证食物安全</li>
<li>科学的烹调加工</li>
<li>合理的进餐制度和良好的饮食习惯</li>
<li>遵循《中国居民膳食指南》的原则</li>
</ol>
<h3 id="蛋白质"><a href="#蛋白质" class="headerlink" title="蛋白质"></a>蛋白质</h3><ul>
<li>必需氨基酸（essential amino acid）： 是指人体内不能合成或合成速度不能满足机体需要，必须从食物中直接获得的氨基酸。<br>“携一两本单色书来” + 组氨酸（婴儿）</li>
</ul>
<ul>
<li>非必需氨基酸：人体可以自身合成，不一定需要从食物中直接供给的氨基酸。</li>
</ul>
<ul>
<li>条件必需氨基酸（conditionally essential amino acid）：<br>某些氨基酸在正常情况下能够在体内合成，为非必需氨基酸；但在某些特定条件下，由于合成能力有限或需要量增加，不能满足机体需要，需要从食物中获取，变成必需氨基酸，即条件必需氨基酸。</li>
</ul>
<ul>
<li>氨基酸模式（amino acid pattern）:<br>蛋白质中各种必需氨基酸的构成比例。 # 将色氨酸含量定为1。</li>
</ul>
<ul>
<li>限制氨基酸（limiting amino acid）：<br>含量相对较低的必需氨基酸称为限制氨基酸。其中含量最低的称为第一限制氨基酸，余者依此类推。</li>
</ul>
<ul>
<li>参考蛋白（reference protein）：<br>指可用来测定其他蛋白质质量的标准蛋白。<br>鸡蛋因其蛋白质与人体蛋白质氨基酸模式最接近，在试验中常以它作参考蛋白。</li>
</ul>
<ul>
<li>优质蛋白质/完全蛋白质：<br>含必需氨基酸种类齐全，氨基酸模式与人体蛋白质氨基酸模式接近的蛋白质，营养价值较高。不仅可以维持成人健康，也可促进儿童生长发育。<br>如蛋、奶、肉、鱼等动物性蛋白质及大豆蛋白等。</li>
</ul>
<ul>
<li>半完全蛋白质：<br>虽含有种类齐全的必需氨基酸但氨基酸模式与人体蛋白质氨基酸模式差异较大，其中一种或几种必需氨基酸相对含量较低，导致其他的必须氨基酸在体内不能被充分利用而被浪费，虽可维持生命但不能促进生长发育的蛋白质。<br>如大多数植物蛋白。</li>
</ul>
<ul>
<li>蛋白质的互补作用（protein complementary action）：<br>为了提高植物性蛋白质的营养价值，将两种或两种以上的食物混合食用，从而达到以多补少，提高膳食蛋白质营养价值的目的。这种食物间互相补充其氨基酸不足的作用称为蛋白质互补作用。</li>
</ul>
<p>蛋白质的功能：</p>
<ol>
<li>人体组织的构成成分：瘦组织，肌肉，骨骼牙齿胶原蛋白，指甲，细胞结构等。</li>
<li>构成体内各种重要的生理活性物质，调节生理功能：酶，激素，抗体，转运体；渗透压，酸碱度。</li>
<li>供给能量</li>
<li>肽类的特殊生理功能：免疫调节，促进矿物质吸收，降血压，清除自由基。</li>
</ol>
<p>食物蛋白质营养学评价<br>（1）蛋白质含量：凯氏定氮法等</p>
<p>（2）蛋白质消化率</p>
<ol>
<li>蛋白质消化率 (digestibility)：<br>蛋白质消化率不仅反应蛋白质在体内被分解的程度，同时还反应消化后氨基酸和肽被吸收程度。</li>
<li>真消化率（true ）：<br>TD=[食物氮-(粪氮-粪代谢氮)]÷食物氮×100%<br>（粪代谢氮，是指肠内源性氮，是在试验对象完全不摄入蛋白质时，粪中含氮量。）</li>
<li>表观消化率（apparent digestibility）：<br>在实际应用中，常不考虑粪代谢氮。这样不仅实验方法简便，且因所测结果比真消化率要低，具有安全性。这种消化率叫做表观消化率。AD=（食物氮-粪氮）÷食物氮×100%</li>
</ol>
<p>（3）蛋白质利用率</p>
<ol>
<li>生物价（Biological value, BV）：<br>蛋白质生物价是反映食物蛋白质消化吸收后，被机体利用程度的指标，生物价的值越高，表明其被机体利用程度越高，最大值为100。</li>
<li>蛋白质净利用率（net protein utilization, NPU）：<br>蛋白质净利用率是反应食物中蛋白质被利用的程度。此项指标包括食物蛋白质消化和利用，故更为全面。<br>蛋白质净利用率(％)=消化率×生物价=储留率÷食物氮×100</li>
<li>蛋白质功效比值(protein efficiency ratio, PER) ：<br>蛋白质功效比值测定，是用处于生长阶段的幼年动物（一般用刚断奶的雄性大白鼠），以其在实验期内体重增加和摄入蛋白质的量的比值，作为反映蛋白质营养价值的指标。<br>蛋白质功效比值=动物体重增加（g）/摄入食物蛋白质（g）</li>
<li>氨基酸评分 (amino acid score, AAS)：<br>是最简单的评估蛋白质质量的方法。该法是用被测食物蛋白质必需氨基酸评分模式和推荐的理想模式或参考蛋白模式进行比较。<br>氨基酸评分=(被测蛋白质每克氮（或蛋白质）中必需氨基酸量（mg）)/(理想模式或参考蛋白中每克氮（或蛋白质）中相应的必需氨基酸量（mg）)×100</li>
<li>经消化率修正的氨基酸评分(protein digestibility corrected amino acid score, PDCAAS）：<br>经消化率修正的氨基酸评分=氨基酸评分×真消化率</li>
<li>其他指标：相对蛋白质值，净蛋白质比值和氮平衡指数等。</li>
</ol>
<ul>
<li>蛋白质-能量营养不良（protein-energy malnutrition, PEM）：</li>
</ul>
<ol>
<li>Kwashiorkor，指能量摄入基本满足而蛋白质严重不足的儿童营养性疾病，主要表现为腹腿部水肿、虚弱、表情淡漠、生长滞缓、头发变色、变脆和易脱落、易感染其他疾病等；</li>
<li>Marasmus，原意即为“消瘦”，指蛋白质和能量摄入均严重不足的儿童营养性疾病，患儿消瘦无力，因易感染其他疾病而死亡。</li>
</ol>
<p>蛋白质摄入过多的危害：</p>
<ol>
<li>过多动物性蛋白质的摄入，必定伴有较多动物脂肪和胆固醇的摄入</li>
<li>加重肾脏负荷</li>
<li>含硫氨基酸摄入过多，加速骨骼中钙的流失，易产生骨质疏松症</li>
<li>同型半胱氨酸可能是心脏疾病的危险因素</li>
</ol>
<p>蛋白质营养状况评价：</p>
<ol>
<li>血清蛋白质</li>
<li>上臂肌围和上臂肌区：评价总体蛋白质储存比较可靠的指标</li>
<li>血清氨基酸比值</li>
</ol>
<ul>
<li>氮平衡(nitrogen balance):<br>摄入蛋白质的量和排出蛋白质的量之间的关系称为氮平衡。<br>氮平衡关系式如下：B=I－(U+F+S)，<br>B：氮平衡；I：摄入量；U：尿素；F：粪氮；S：皮肤等氮损失。</li>
</ul>
<h3 id="脂肪"><a href="#脂肪" class="headerlink" title="脂肪"></a>脂肪</h3><p>脂质的功能</p>
<p>（1）体内脂肪的生理功能：</p>
<ol>
<li>贮存和提供能量</li>
<li>保温及润滑作用</li>
<li>节约蛋白质作用</li>
<li>机体构成成分：细胞膜</li>
<li>内分泌作用：瘦素、TNF-α、IL-6、IL-8、雌激素、IGF-1等。</li>
</ol>
<p>（2）食物中脂肪的作用：<br>提供能量，作为人体脂肪的合成材料</p>
<ol>
<li>增加饱腹感  脂肪进入十二指肠时，刺激十二指肠产生肠抑胃素，使胃蠕动受到抑制。</li>
<li>改善食物的感观性状</li>
<li>提供脂溶性维生素：脂肪不仅是脂溶性维生素ADEK来源，也可促进其吸收。</li>
</ol>
<p>（3)类脂的功能（磷脂 &amp; 胆固醇）</p>
<ol>
<li>提供能量</li>
<li>细胞膜成分：双重特性，利于细胞内外物质交换。</li>
<li>乳化剂功能：脂肪悬浮，促进脂肪吸收、转运、代谢。</li>
<li>改善心血管功能：防止胆固醇沉积、降低血黏度，预防心血管疾病。</li>
<li>改善神经系统功能：释放胆碱，乙酰胆碱，促进和改善大脑组织和神经系统的功能</li>
<li>胆固醇是细胞膜的重要成分</li>
<li>胆固醇可合成激素和VitD3等。</li>
</ol>
<ul>
<li>必需脂肪酸(essential fatty acid, EFA）：<br>人体不可缺少且自身不能合成，必须通过食物供给的脂肪酸。<br>EFA包括亚油酸（n-6）和α-亚麻酸（n-3）。</li>
</ul>
<ol>
<li>构成磷脂的组成成分：构成细胞膜</li>
<li>前列腺素合成的前体：血管舒缩、神经传导等</li>
<li>参与胆固醇代谢：体内约70%胆固醇与脂肪酸酯化</li>
</ol>
<p>膳食脂肪的营养学评价</p>
<ol>
<li>脂肪的消化率：熔点</li>
<li>必需脂肪酸含量</li>
<li>各种脂肪酸的比例</li>
<li>脂溶性维生素含量</li>
</ol>
<h3 id="碳水化合物"><a href="#碳水化合物" class="headerlink" title="碳水化合物"></a>碳水化合物</h3><ul>
<li>膳食纤维（dietary fiber）：<br>主要包括纤维素、木质素、抗性低聚糖、果胶、抗性淀粉以及其他不可消化的碳水化合物。<br>主要来源于全谷类、蔬菜水果等<br>生理学意义：促进肠道健康功能。（见碳水化合物的功能第5条）</li>
</ul>
<ul>
<li>益生元：<br>指不被人体消化系统消化和吸收，能够选择性地促进宿主肠道内原有的一种或几种有益细菌（益生菌）生长繁殖的物质。通过有益菌的繁殖增多，抑制有害细菌生长，从而达到调整肠道菌群的目的，促进机体健康的目的。<br>如乳果糖和异麦芽低聚糖。</li>
</ul>
<ul>
<li>食物血糖生成指数 （glycemic index，GI）：<br>简称生糖指数，是反映食物引起人体血糖升高程度的指标，是人体进食后机体血糖生成的应答状况。<br>GI=某事物摄入后2小时血糖曲线下面积 / 等量葡萄糖摄入后2小时血糖曲线下面积 × 100%<br>可以衡量某种食物或某种膳食组成对血糖浓度的影响。</li>
</ul>
<ul>
<li>血糖负荷（glycemic load, GL）：<br>评价某种食物摄入量对人体血糖影响的程度。<br>GL= 摄入食品中碳水化合物的重量×食品的GI值/100</li>
</ul>
<p>碳水化合物的功能</p>
<ol>
<li>提供能量</li>
<li>构成组织结构及生理活性物质</li>
<li>血糖调节作用</li>
<li>节约蛋白质作用和抗生酮作用</li>
<li>膳食纤维的促进肠道健康功能 <ol>
<li>增加饱足感：吸收水份，增加胃内的食物体积</li>
<li>促进排便：吸水增加粪便体积，机械刺激肠壁蠕动</li>
<li>降低血糖和血胆固醇：减少小肠对糖的吸收，吸附肠道中脂质降低吸收率</li>
<li>改变肠道菌群：促进某些有益菌群的增殖</li>
</ol>
</li>
</ol>
<h3 id="能量"><a href="#能量" class="headerlink" title="能量"></a>能量</h3><ul>
<li>能量系数/热价（calorific coefficient/calorific value）：<br>每克宏量营养素（产能营养素：碳水化合物、脂肪和蛋白质）在体内氧化分解（或在体外燃烧）所产生的能量值。</li>
</ul>
<p>人体的能量消耗</p>
<ol>
<li>基础代谢（basal metabolism）：<br>又称基础能量消耗（BEE）：用于维持机体最基本的生命活动所需要的能量消耗，占人体总能量消耗的60~70%</li>
</ol>
<ul>
<li>基础代谢率（basal metabolic rate，BMR）：每小时每平方米体表面积（或每千克体重）的基础能量消耗<br>BEE的影响因素：体型与体质、生理和病理状况、生活和作业环境</li>
</ul>
<ol start="2">
<li>身体活动（physical activity）：<br>是指任何由骨骼肌收缩引起能量消耗的身体运动，约占人体总能量消耗的15~30%<br>影响因素：肌肉发达、体重越重、工作越不熟练者，消耗能量越多</li>
<li>食物热效应（thermic effect of food, TEF）：<br>人体在摄食过程中所引起的额外能量消耗，是摄食后发生的一些列消化、吸收活动以及营养素和其代谢产物之间相互转化过程所消耗的能量，又称食物特殊动力作用（specific dynamic action, SDA）</li>
<li>特殊生理阶段的能量消耗：孕期组织生长及儿童生长发育</li>
</ol>
<ul>
<li>能量需要量（estimated energy requirement, EER）：<br>是指能长期保持良好的健康状态，维持良好的体型、机体构成和理想活动水平的个人或人群，达到能量平衡时所需要的膳食能量摄入量。可以直接等同于相应人群的能量平均需要量（EAR）。</li>
</ul>
<p>能量摄入的调节</p>
<ol>
<li>神经生理对摄食的调节：通过摄食系统和饱食系统调节摄食启动和终止</li>
<li>营养素及其代谢产物对摄食的调节：调控摄食信号因子和饱食信号因子</li>
<li>蛋白和肽类因子对摄食的调节：组织细胞中生长素释放肽、胰多肽，瘦素、胆囊收缩素分别促进和抑制食欲和能量代谢；神经肽Y，食欲肽A、B促进和抑制食欲和能量代谢；饱腹因子，促肾上腺皮质激素释放激素抑制摄食行为</li>
<li>蛋白因子对能量消耗的影响：解耦联蛋白，β3-肾上腺素受体促进产热和能量消耗</li>
<li>非生理因素对能量摄入的影响：进食环境，食物特性，饮食习惯，食物信念和态度，社会文化因素等</li>
</ol>
<h3 id="矿物质"><a href="#矿物质" class="headerlink" title="矿物质"></a>矿物质</h3><p>矿物质的特点</p>
<ol>
<li>矿物质在体内不能合成，必须从外界摄取；</li>
<li>矿物质是唯一可以通过天然水途径获取的营养素；</li>
<li>矿物质在体内分布极不均匀；</li>
<li>矿物质之间存在协同或拮抗作用；</li>
<li>某些微量元素在体内的生理剂量与中毒剂量范围较窄，摄入过多易产生毒性作用。</li>
</ol>
<p>人体矿物质的缺乏与过量的原因？P79</p>
<ol>
<li>地球环境因素：如地方病</li>
<li>食物成分及加工因素；</li>
<li>人体自身因素：摄入不足、需求增加、疾病状态</li>
</ol>
<h3 id="维生素"><a href="#维生素" class="headerlink" title="维生素"></a>维生素</h3><ul>
<li>维生素（Vitamin）：<br>是维持机体活动过程所必需的一类微量低分子有机物，在机体物质和能量代谢过程中其重要作用。</li>
</ul>
<h2 id="各类食品的营养价值"><a href="#各类食品的营养价值" class="headerlink" title="各类食品的营养价值"></a>各类食品的营养价值</h2><ul>
<li><p>食物的营养价值（nutritional value）：某种食物所含营养素和能量能满足人体营养需要的程度。食物营养价值的高低取决于食品中营养素的种类是否齐全、数量的多少、相互比例是否适宜以及是否被人体消化吸收和利用。</p>
</li>
<li><p>营养质量指数（index of nutrition quality，INQ）是指食物中某营养素能满足人体营养需要的程度（营养素密度）与该食物能满足人体能量需要的程度（能量密度）的比值。公式如下：</p>
</li>
</ul>
<p>营养调查：系指运用各种手段准确地了解某人群或特定个体各种营养指标的水平，以判断其当前的营养和健康状况，是公共营养的基本方法和内容。（P200）<br>    24小时食物回顾法：即24小时膳食的回顾调查，对被调查者24小时内各种主副食物摄入情况进行回顾调查连续3天，获得个人每日各种食物摄入量。根据食物成分表计算出摄入的能量和营养素<br>    营养监测：是指长期动态监测人群的营养状况，同时收集影响人群营养状况的有关社会经济等方面的资料，探讨从政策上、社会措施上改善营养状况和条件的途径。（P204）</p>
<h2 id="特殊人群营养"><a href="#特殊人群营养" class="headerlink" title="特殊人群营养"></a>特殊人群营养</h2><h2 id="公共营养"><a href="#公共营养" class="headerlink" title="公共营养"></a>公共营养</h2><h1 id="食品卫生学"><a href="#食品卫生学" class="headerlink" title="食品卫生学"></a>食品卫生学</h1><ul>
<li><p>食品安全 # 过程 &amp; 结果安全<br>指食品无毒、无害，符合应当有的营养要求，对人体健康不造成任何急性、亚急性或慢性危害。</p>
</li>
<li><p>食品卫生 # 过程安全<br>是为防止食品在生产、收获、加工、运输、储存、销售等各个环节被有害物质（包括物理、化学、微生物等方面）污染所采取的各项措施，从而保证人体健康不受损害。</p>
</li>
</ul>
<h2 id="食品污染及其预防"><a href="#食品污染及其预防" class="headerlink" title="食品污染及其预防"></a>食品污染及其预防</h2><ul>
<li>食品污染（Food contamination）：<br>是指各种条件下，导致外源性有毒有害物质进入食品，或食物成分本身发生化学反应而产生有毒有害物质，从而造成食品安全性、营养性和（或）感官性状发生改变的过程。</li>
</ul>
<ul>
<li>水分活度（water activity，Aw）：<br>食物中微生物可利用的水,指食品中水蒸气分压（P）与同条件下纯水的蒸气压（P0）之比（Aw=P/P0），值越小越不利于微生物增殖。</li>
</ul>
<ul>
<li>黄曲霉毒素（aflatoxin, AF）：<br>是黄曲霉和寄生曲霉产生的一类代谢产物，具有极强的毒性和致癌性。</li>
</ul>
<p>黄曲霉毒素的预防措施：</p>
<ol>
<li>食物防霉：根本措施，可采用物理、化学及生物等方法。</li>
<li>去除毒素：①拣霉粒法 ②碾轧法 ④植物油加碱炼法 ⑤生物方法：分解、转化、去除AFB1  </li>
<li>制定食物中AF限量标准 </li>
</ol>
<ul>
<li>食品腐败变质（food spoilage）：<br>是指食品在以微生物为主的各种因素作用下，其原有的化学性质或物理性质，即食品成分与感官性状的各种变化，降低或失去营养价值的过程。</li>
</ul>
<ul>
<li>挥发性盐基总氮（total volatile basic nitrogen, TVBN）：<br>指食品水浸液在碱性条件下能与水蒸气一起蒸馏出来的总氮量，即在此条件下能形成的氨的含氮物。</li>
</ul>
<ul>
<li>K值（K value）：</li>
</ul>
<ul>
<li>N-亚硝基化合物（N-nitroso compounds, NOCs）：<br>是一类具有&gt;N-N=O结构的有机化合物，暴露源广泛，多具有遗传毒性和动物致癌性。</li>
</ul>
<h2 id="食品添加剂及其管理"><a href="#食品添加剂及其管理" class="headerlink" title="食品添加剂及其管理"></a>食品添加剂及其管理</h2><ul>
<li>食品添加剂：<br>为改善食品品质和色、香、味，以及防腐、保鲜和加工工艺需要而加入食品中的人工合成或天然物质。</li>
</ul>
<p>食品添加剂使用的基本要求</p>
<ol>
<li>不应对人体产生任何健康危害；</li>
<li>不应掩盖食品腐败变质；</li>
<li>不应掩盖食品本身或加工过程中的质量缺陷，或以掺杂、掺假、伪造为目的而使用食品添加剂；</li>
<li>不应降低食品本身的营养价值；</li>
<li>在达到预期效果的前提下尽可能降低在食品中的用量：</li>
</ol>
<h2 id="各类食品卫生及其管理"><a href="#各类食品卫生及其管理" class="headerlink" title="各类食品卫生及其管理"></a>各类食品卫生及其管理</h2><h2 id="食源性疾病及其预防"><a href="#食源性疾病及其预防" class="headerlink" title="食源性疾病及其预防"></a>食源性疾病及其预防</h2><ul>
<li>食源性疾病（foodborne disease）：<br>通过摄食进入人体内的各种致病因子引起的、通常具有感染性质或中毒性质的一类疾病。</li>
</ul>
<ul>
<li>人兽共患传染病（anthropzoonoses）：指人和脊椎动物之间自然感染与传播的疾病。</li>
</ul>
<ul>
<li>食物中毒（food poisoning）：<br>指摄入含有生物性、化学性有毒有害物质的食物或把有毒有害物质当作食物摄入后出现的非传染性急性或亚急性疾病。</li>
</ul>
<ul>
<li>食物过敏：（food allergy）：<br>食物中的抗原诱导机体免疫应答而发生的变态反应性疾病。</li>
</ul>
<h2 id="食品安全性风险分析和控制"><a href="#食品安全性风险分析和控制" class="headerlink" title="食品安全性风险分析和控制"></a>食品安全性风险分析和控制</h2><h2 id="食品安全监督管理"><a href="#食品安全监督管理" class="headerlink" title="食品安全监督管理"></a>食品安全监督管理</h2><p>食品安全法律法规体系构成</p>
<ol>
<li>食品安全法律：宪法，一般法（刑法，民法，刑事、民事、行政诉讼法），专门法（食品安全法）；</li>
<li>食品安全法规：行政法规，地方法规；</li>
<li>食品安全规章：部门规章，地方规章；</li>
<li>食品安全标准；</li>
<li>其他规范性文件：地方食药和卫生部分的管理办法和规定等；</li>
</ol>
<ul>
<li>食品安全标准：<br>是指对食品中具有与人类健康相关的质量要素和技术要求及其检验方法、评价程序等所作的规定。</li>
</ul>
<p>食品安全标准的性质</p>
<ol>
<li>政策法规性</li>
<li>科学技术性</li>
<li>强制性</li>
<li>社会性</li>
<li>经济性</li>
</ol>
<p>食品安全标准的意义</p>
<ol>
<li>食品安全标准是食品安全法律法规体系的重要组成部分</li>
<li>食品安全标准是食品安全法制化管理的重要依据</li>
<li>食品安全标准是维护国家主权、促进食品国际贸易的技术保障。</li>
</ol>
<p>按标准发生作用的范围或其审批权限的分类</p>
<ol>
<li>食品安全国家标准</li>
<li>食品安全地方标准</li>
<li>企业标准</li>
</ol>
<p>食品安全标准的制定依据</p>
<ol>
<li>法律依据：《食品安全法》和《标准化法》；</li>
<li>与国际标准协调一致性</li>
<li>科学技术依据</li>
</ol>
<p>食品安全标准的主要技术指标</p>
<ol>
<li>严重危害人体健康的指标：致病微生物与毒素，有毒有害化学物质，放射性污染物等；</li>
<li>反映食品可能被污染及污染程度的指标：菌落总数，大肠菌群等；</li>
<li>间接反映食品安全质量发生变化的指标：水分，含氮化合物，挥发性盐基总氮等；</li>
<li>营养指标：各种营养素含量；</li>
<li>商品质量指标</li>
</ol>
<p>食品中有毒有害物质限量标准的指定</p>
<ol>
<li>确定动物最大无作用剂量：NOAEL；</li>
<li>确定人体可耐受摄入量</li>
<li>确定每日总膳食中的允许含量</li>
<li>确定每种食物肿的最大允许量</li>
<li>指定食品中有毒有害物质的限量标准</li>
</ol>
<p>食品安全监督管理的原则：</p>
<ol>
<li>预防为主</li>
<li>风险管理</li>
<li>全程控制</li>
<li>社会共治</li>
</ol>
<ul>
<li><p>食品良好生产规范（good manufacture practice, GMP）：<br>是为保障食品安全、质量而制定的贯彻食品生产全过程的一系列措施、方法和技术要求。</p>
</li>
<li><p>危害分析关键控制点（HACCP）：<br>是一种食品安全保障体系，为保障食品安全，对食品生产加工过程中造成食品污染发生或发展的各种危害因素进行系统和全面的分析，确定能够有效预防、减轻或消除危害的加工环节（即“关键控制点”），进而在关键控制点对危害因素进行控制，同时监测控制效果，发生偏差时予以纠正，并随时对控制方法进行矫正和补充。（P515）</p>
</li>
</ul>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/15/SIMCA_in_action/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/15/SIMCA_in_action/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-15T23:31:43+08:00">
                2018-04-15
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="SIMCA官方的NMR代谢组学教程实战"><a href="#SIMCA官方的NMR代谢组学教程实战" class="headerlink" title="SIMCA官方的NMR代谢组学教程实战"></a>SIMCA官方的NMR代谢组学教程实战</h1><h1 id="用正交偏最小二乘法判别分析处理核磁共振代谢组学数据"><a href="#用正交偏最小二乘法判别分析处理核磁共振代谢组学数据" class="headerlink" title="用正交偏最小二乘法判别分析处理核磁共振代谢组学数据"></a>用正交偏最小二乘法判别分析处理核磁共振代谢组学数据</h1><p>华中科技大学同济医学院公共卫生学院 夏鹏飞<br><a href="mailto:U201412842@hust.edu.cn" target="_blank" rel="noopener">U201412842@hust.edu.cn</a><br><a href="https://github.com/XIAPengfei1996" target="_blank" rel="noopener">https://github.com/XIAPengfei1996</a></p>
<h2 id="代谢组学数据分析简介"><a href="#代谢组学数据分析简介" class="headerlink" title="代谢组学数据分析简介"></a>代谢组学数据分析简介</h2><p>代谢组学几乎完全不涉及生物信息学最核心的序列比对，包括武汉中科院数物所波谱国重实验室等主流科研机构都是利用化学计量学和多元统计分析方法，对通过核磁共振（NMR）、液相色谱质谱联用（LC-MS）、气相色谱质谱联用（GC-MS）等手段获取的小分子代谢物（相对分子质量MW小于1000）的定量数据进行处理。除此之外只有一些商业公司在做外包服务，因此相比基因组、转录组、蛋白质组显得非常小众。</p>
<p>目前在代谢组学中运用较多的多元统计分析方法，包括主成分分析（Principal component analysis, PCA）、层次聚类分析（Hierarchical clustering analysis, HCA）、非线性映射（Nonlinear innuendo analysis, NLM）、分级聚类法（hierarchical cluster analysis, HCA）等非监督分类方法，以及K最邻近法（K-nearest neighbor classification method, K-NN）、偏最小二乘法-判别分析（Partial least squares projections to latent structures - discriminant analysis, PLS-DA）、正交偏最小二乘法-判别分析（Orthogonal projections to latent structures - discriminant analysis, OPLS-DA）、人工神经网络（Artificial Neural Network, ANN）、支持向量机（Support Vector Machine, SVM）等监督分类方法。篇幅所限，有关这些方法的具体内容和意义请参考相关文献。</p>
<p>代谢组学数据分析最常用的软件SIMCA是1987年由瑞典Umetrics公司研究开发的一款多元变量统计分析软件。官方网站随便一个课程视频都是一、两千美刀，其他学习资源目前也都限于各生物公司的培训资料，同样价格不菲。这里我利用SIMCA官方的教程和数据操作一遍完整的流程以飨读者。</p>
<h2 id="下载并安装软件"><a href="#下载并安装软件" class="headerlink" title="下载并安装软件"></a>下载并安装软件</h2><ul>
<li><a href="http://landing.umetrics.com/downloads-simca" target="_blank" rel="noopener">官方网站的下载页面</a>提供了30天免费试用版SIMCA，当前更新到第15版；下载解压得到.exe格式文件，安装后即可在Windows系统上运行。由于之前为了给转录组入门实战数据腾位置，我删掉了苹果机上的Windows双系统，这次我将临时在Parallels Desktop 13试用版虚拟机上运行Windows 10系统来进行操作。</li>
</ul>
<h2 id="教程及数据"><a href="#教程及数据" class="headerlink" title="教程及数据"></a>教程及数据</h2><ul>
<li><a href="http://landing.umetrics.com/downloads-simca" target="_blank" rel="noopener">官网下载页</a>虽然也提供了官方教材<a href="https://github.com/XIAPengfei1996/Shia/raw/master/SIMCA/Analysing_NMR_metabolomics_data_using_OPLS-DA.pdf" target="_blank" rel="noopener">Analysing_NMR_metabolomics_data_using_OPLS-DA.pdf</a>，但是和数据<a href="https://github.com/XIAPengfei1996/Shia/raw/master/SIMCA/NMR%20METABOLOMICS.xls" target="_blank" rel="noopener">NMR_METABOLOMICS.xls</a>一样都非常不便下载，不过好在网上随便搜索一下也能找到很多流传的资源。方便起见，我上传到了我的GitHub里面，点击文件标题名就可以直接下载。</li>
</ul>
<h2 id="数据信息"><a href="#数据信息" class="headerlink" title="数据信息"></a>数据信息</h2><p>该研究以杨树的文库中编码功能未知的转录因子MYB的基因PttMYB76为例，比较MYB76杨树与野生型杨树生长过程的代谢组学特征差异。实验材料包括来自MYB76杨树（编号为Ai, Bi, Ci）和野生型杨树（编号为Di, Ei, Fi）两组共6棵树的45个样品，其中i代表生长方向，即1代表顶部，8代表底部。加上12组重复观测值，数据集中总共有57组观测值，655种变量。由于这里我们主要目的是做数据分析，因此对实验细节不加赘述，有需要者可以参考原文：<br><a href="https://eurekamag.com/pdf.php?pdf=004019559" target="_blank" rel="noopener">S. Wiklund et.al A new metabonomic strategy for analysing the growth process of the poplar tree. Plant Biotechnology Journal 2005 3 pp 353-362</a><br><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/1_excel.png" alt="1_excel"></p>
<h2 id="数据导入"><a href="#数据导入" class="headerlink" title="数据导入"></a>数据导入</h2><p>打开菜单栏中的<code>File</code>，点击<code>New</code>选项中的<code>Regular project</code>弹出浏览界面，选择点击下载好的数据集<code>NMR_METABOLOMICS.xls</code>文件导入数据。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/2_file.png" alt="2_file"></p>
<p>这里还需要按照软件要求调整数据格式，打开菜单栏中的<code>Edit</code>，点击<code>Transpose</code>将表格翻转90度；</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/3_transpose.png" alt="3_transpose"></p>
<p>在打开菜单栏中的<code>Home</code>，依次选择第一行后点击<code>Primary variable ID</code>，选择第二列后点击<code>Secondary observation ID</code>；最后点击右上角的<code>Finish import</code>，保存后即完成数据导入。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/4_import.png" alt="4_import"></p>
<h2 id="主成分分析"><a href="#主成分分析" class="headerlink" title="主成分分析"></a>主成分分析</h2><p>打开新建的模型M1，点击模型界面左上角的<code>Workset…</code>，选择<code>Scale</code>界面，点击左下角<code>Find Primary ID…</code>右侧的三角形，选择第一行的<code>Select all</code>（或者<code>Ctrl</code>+<code>A</code>）全选变量名；然后点击右侧<code>Type</code>项中下拉按钮，选择<code>Par</code>选项后，点击下方的<code>Set all</code>，对所有变量值进行Pareto标度化。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/6_scale.png" alt="6_scale"></p>
<p>再选择<code>Observations</code>界面，长按<code>Ctrl</code>键，选出A、B、C三棵MYB76杨树对应观测值，在<code>Set class</code>栏中输入2，点击<code>Set</code>将MYB76杨树观测值编为2组；同理将D、E、F三棵野生型杨树对应观测值编为1组；再在下方的<code>Model typr</code>栏中选择<code>PCA-X</code>，点击<code>确定</code>完成设置。这里要求处理组样本需要具有较高的类别编号，对照样本需要具有较低的类别编号，这样做会有利于后面的直接解释。 </p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/7_observations.png" alt="7_observations"></p>
<p>选择新建的M1模型，点击上方工具栏中的<code>Autofit</code>自动拟合最佳模型。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/8_autofit.png" alt="8_autofit"></p>
<p>生成的模型产生的8个主成分，这里我们看到第8主个成分增加后，模型预测性Q2没有明显增加，说明这个主成分对于模型的贡献以及非常微小。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/9_fit.png" alt="9_fit"></p>
<p>为了便于分析，这里我们只选取前3个主成分，可以通过点击<code>Autofit</code>右侧的<code>remove</code>删去多余的主成分。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/10_remove.png" alt="10_remove"></p>
<p>如果误删成分，可以点击<code>remove</code>上方的<code>add</code>恢复。（我没有点错！我只是为了给大家试错，对，没错，就是这样！）<br>最后的结果应该是这样：</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/11_component.png" alt="11_component"></p>
<p>点击上方工具栏中的<code>DModX</code>可以选择生成折线图或直方图查看离群值。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/12_DModX.png" alt="12_DModX"></p>
<p>点击上方工具栏中的<code>Scores</code>可以选择<code>Scatter</code>生成以第一主成分t1和第二主成分t2分别为横纵坐标的t1-t2得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/13_scoreplott1t2.png" alt="13_scoreplott1t2"></p>
<p>也可以选择<code>3D</code>生成分别以三个主成分为坐标的三维得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/14_scoreplot3d.png" alt="14_scoreplot3d"></p>
<p>这里两组样本数据并没有得到很好的区分，我们需要在看一下t2-t3得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/15_scoreplott2t3.png" alt="15_scoreplott2t3"></p>
<p>实现这个得分图需要点击工具栏中的<code>Scores</code>选项下最右侧的图标，或者右键点击之前生成的得分图，点击第二个图标，进入设置界面。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/16_dialog.png" alt="16_dialog">、</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/17_properties.png" alt="17_properties"></p>
<p>在<code>Scores</code>页面中<code>Selected</code>栏下，<code>X-axis</code>对应的<code>Comp</code>选择<code>2</code>，<code>Series</code>对应的<code>Comp</code>选择<code>3</code>，点击<code>确定</code>即可生成t2-t3得分图.</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/18_scores.png" alt="18_scores"></p>
<p>这里我们对比两个得分图可以明显看出，后者对两组样本数据的区分效果优于前者。</p>
<p>此外，如果仔细观察t1-t2得分图，还可以发现生长方向i值有明显地沿某一方向依次变化的趋势，表现了杨树生长过程的代谢组学特征的规律性变化。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/19_trend.png" alt="19_trend"></p>
<h2 id="正交偏最小二乘法判别分析"><a href="#正交偏最小二乘法判别分析" class="headerlink" title="正交偏最小二乘法判别分析"></a>正交偏最小二乘法判别分析</h2><p>右键点击模型M1，选择<code>New as Model 1</code>生成新的模型M2。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/20_newasmedel.png" alt="20_newasmedel"></p>
<p>选择<code>Observations</code>界面，长按<code>Ctrl</code>键，选出最后12组重复观测值，右键选择<code>Exclude</code>；再在下方的<code>Model typr</code>栏中选择<code>OPLS-DA</code>，点击<code>确定</code>完成设置。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/21_exclude.png" alt="21_exclude"></p>
<p>再次点击<code>Autofit</code>自动拟合最佳模型，得到一个1+4的OPLS模型。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/22_component.png" alt="22_component"></p>
<p>点击上方工具栏中的<code>Scores</code>，选择<code>Scatter</code>即可生成OPLS-DA得分图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/23_oplsda.png" alt="23_oplsda"></p>
<h2 id="主成分分析和正交偏最小二乘法判别分析的对比"><a href="#主成分分析和正交偏最小二乘法判别分析的对比" class="headerlink" title="主成分分析和正交偏最小二乘法判别分析的对比"></a>主成分分析和正交偏最小二乘法判别分析的对比</h2><p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/pca_vs_oplsda.png" alt="pca_vs_oplsda"></p>
<p>显然正交偏最小二乘法判别分析的分类效果更佳！</p>
<h2 id="解释正交偏最小二乘法判别分析模型"><a href="#解释正交偏最小二乘法判别分析模型" class="headerlink" title="解释正交偏最小二乘法判别分析模型"></a>解释正交偏最小二乘法判别分析模型</h2><p>点击上方工具栏中的<code>Loadings</code>，选择<code>Line</code>生成图片后，右键点击图片，点击<code>Properties</code>，在<code>Loadings</code>页面中选择<code>Selected</code>栏下，<code>X-axis</code>对应的<code>Item</code>选择<code>Var ID</code>，<code>Series</code>对应的<code>Item</code>选择<code>pq</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/24_loadings.png" alt="24_loadings"></p>
<p>点击<code>确定</code>生成载荷图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/25_loadingline.png" alt="25_loadingline"></p>
<p>点击上方菜单栏中的<code>Plot/List</code>，点击工具栏中的<code>Scatter</code>，在<code>Data series</code>页面中<code>Select data type</code>项选择<code>Observations and loadings</code>，<code>X-axis</code>对应的<code>Item</code>选择<code>p</code>，<code>Series</code>对应的<code>Item</code>选择<code>p(corr)</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/26_scatter.png" alt="26_scatter"></p>
<p>点击<code>确定</code>生成S-plot。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/27_splot.png" alt="27_splot"></p>
<p>标记点为可能的生物标志物。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/28_loading&amp;s.png" alt="28_loading&amp;s"></p>
<h2 id="诊断正交偏最小二乘法判别分析模型"><a href="#诊断正交偏最小二乘法判别分析模型" class="headerlink" title="诊断正交偏最小二乘法判别分析模型"></a>诊断正交偏最小二乘法判别分析模型</h2><p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/diagnostics.png" alt="diagnostics"></p>
<p><strong>模型概要</strong></p>
<p>R2X(cum)是模型所解释的X预测和正交变异之和，即0.157+0.613=0.769，也可以被解释为X总变异的76.9%；</p>
<p>R2Y(cum) 是模型所解释Y的总变异, 此处为0.977；</p>
<p>Q2(cum)衡量模型预测好坏，此处为0.941；</p>
<p><strong>预测变异 = X变异中与Y相关的部分</strong></p>
<p>A代表于X和Y之间的相关主成分的数量。如果仅使用一个响应向量，则A总是1；</p>
<p>R2X衡量X变异中与Y相关部分，这里是0.157；</p>
<p><strong>正交变异 = X变异中与Y无关的部分</strong></p>
<p>A在此对应于不相关（正交）主成分的数量。每个正交分量都被表示出来并且可以被单独解释；</p>
<p>R2X是衡量X变异中与Y不相关的部分。每个主成分的信息都被分别表示出来；</p>
<p>首行中粗体的R2X（cum）是X变异中与Y无关部分的总和，此处为0.613；</p>
<p>噪声 = 1- 0.157 – 0.613 = 0.23 = 23%</p>
<h2 id="补充"><a href="#补充" class="headerlink" title="_补充_"></a>_补充_</h2><h3 id="响应排序检验"><a href="#响应排序检验" class="headerlink" title="响应排序检验"></a>响应排序检验</h3><p>点击上方菜单栏中的<code>Analyze</code>，点击工具栏中的<code>Permutations</code>，在<code>Permutation Plot</code>页面中<code>Number of permutations to</code>项选择<code>999</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/29_permutatuons.png" alt="29_permutatuons"></p>
<p>点击<code>确定</code>生成响应排序检验示意图，通常R2Y和Q2与直线的斜率越接近水平，模型越有可能过拟合。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/30_permutatuon.png" alt="30_permutatuon"></p>
<h3 id="变量权重重要性排序"><a href="#变量权重重要性排序" class="headerlink" title="变量权重重要性排序"></a>变量权重重要性排序</h3><p>点击工具栏中的<code>VIP</code>，选择<code>VIP predictive</code>。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/31_vip.png" alt="31_vip"></p>
<p>即直接生成变量权重重要性排序图。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/32_vippredictive.png" alt="32_vippredictive"></p>
<p>右键点击图片，选择<code>Create</code>栏中<code>List</code>，可生成变量权重重要性排序表，可以选择VIP值大于1的作为感兴趣的物质进行代谢通路分析。</p>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/SIMCA/33_list.png" alt="33_list"></p>
<h2 id="参考书目"><a href="#参考书目" class="headerlink" title="参考书目"></a>参考书目</h2><p>许国旺. 代谢组学:方法与应用[M]. 科学出版社, 2008.<br>贾伟. 医学代谢组学[M]. 上海科学技术出版社, 2011.</p>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/12/hello-world/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/12/hello-world/" itemprop="url">Hello World</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-12T22:24:12+08:00">
                2018-04-12
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <p>Welcome to <a href="https://hexo.io/" target="_blank" rel="noopener">Hexo</a>! This is your very first post. Check <a href="https://hexo.io/docs/" target="_blank" rel="noopener">documentation</a> for more info. If you get any problems when using Hexo, you can find the answer in <a href="https://hexo.io/docs/troubleshooting.html" target="_blank" rel="noopener">troubleshooting</a> or you can ask me on <a href="https://github.com/hexojs/hexo/issues" target="_blank" rel="noopener">GitHub</a>.</p>
<h2 id="Quick-Start"><a href="#Quick-Start" class="headerlink" title="Quick Start"></a>Quick Start</h2><h3 id="Create-a-new-post"><a href="#Create-a-new-post" class="headerlink" title="Create a new post"></a>Create a new post</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo new <span class="string">"My New Post"</span></span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/writing.html" target="_blank" rel="noopener">Writing</a></p>
<h3 id="Run-server"><a href="#Run-server" class="headerlink" title="Run server"></a>Run server</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo server</span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/server.html" target="_blank" rel="noopener">Server</a></p>
<h3 id="Generate-static-files"><a href="#Generate-static-files" class="headerlink" title="Generate static files"></a>Generate static files</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo generate</span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/generating.html" target="_blank" rel="noopener">Generating</a></p>
<h3 id="Deploy-to-remote-sites"><a href="#Deploy-to-remote-sites" class="headerlink" title="Deploy to remote sites"></a>Deploy to remote sites</h3><figure class="highlight bash"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">$ hexo deploy</span><br></pre></td></tr></table></figure>
<p>More info: <a href="https://hexo.io/docs/deployment.html" target="_blank" rel="noopener">Deployment</a></p>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/Methylation450K_BeadChip/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/Methylation450K_BeadChip/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/ChIP-seq/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/ChIP-seq/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/bioinfonote/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/bioinfonote/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学笔记"><a href="#生物信息学笔记" class="headerlink" title="生物信息学笔记"></a>生物信息学笔记</h1><p><a href="https://www.zhihu.com/question/20543692" target="_blank" rel="noopener">如何自学生物信息学？</a><br><a href="https://www.zhihu.com/question/65066161" target="_blank" rel="noopener">生物信息学入门需要具备什么能力？</a></p>
<p><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0167779915002735-gr1.jpg" alt="Image of Interaction Network"><br>(Source: Yugi K, et al. Trends Biotechnol, 2016)</p>
<h2 id="生物信息学-导论与方法-北京大学"><a href="#生物信息学-导论与方法-北京大学" class="headerlink" title="生物信息学: 导论与方法 | 北京大学"></a>生物信息学: 导论与方法 | 北京大学</h2><p>高歌  魏丽萍  龙漫远  <em>Michael.S.Waterman  Maynard Olson  裴钢</em></p>
<blockquote>
<p>“Essentially, all models are wrong, but some are useful.”<br>—— George E. P. Box</p>
</blockquote>
<p><a href="https://www.coursera.org/learn/sheng-wu-xin-xi-xue" target="_blank" rel="noopener">Coursera</a>  <a href="http://www.chinesemooc.org/mooc/4393" target="_blank" rel="noopener">华文慕课</a>  <a href="http://mib.cbi.pku.edu.cn" target="_blank" rel="noopener">Methods in Bioinformatics</a></p>
<ul>
<li>生信历史</li>
<li>Needleman-Wunsch全局比对算法</li>
<li>Smith-Waterman局部比对算法</li>
<li>BLAST算法</li>
<li>马尔科夫链</li>
<li>隐马尔科夫模型</li>
<li>序列回帖 <code>reads mapping</code></li>
<li>SIFT算法</li>
<li>PolyPhen算法</li>
<li>SAPRED算法</li>
<li>RNAseq回帖</li>
<li>ncRNA鉴定、差异&amp;聚类分析</li>
<li>本体论 <code>ontology</code></li>
<li>KOBAS分子通路鉴定</li>
<li>演化分析</li>
<li>各种测序原理及仪器介绍</li>
<li>各种数据库，软件、网络工具介绍</li>
</ul>
<figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br><span class="line">43</span><br><span class="line">44</span><br><span class="line">45</span><br><span class="line">46</span><br></pre></td><td class="code"><pre><span class="line">&gt;sp|P69905|HBA_HUMAN</span><br><span class="line">MVLSPADKTN VKAAWGKVGA HAGEYGAEAL ERMFLSFPTT KTYFPHFDLS</span><br><span class="line">HGSAQVKGHG KKVADALTNA VAHVDDMPNA LSALSDLHAH KLRVDPVNFK</span><br><span class="line">LLSHCLLVTL AAHLPAEFTP AVHASLDKFL ASVSTVLTSK YR</span><br><span class="line"></span><br><span class="line">&gt;sp|P60871|HBB_HUMAN</span><br><span class="line">MVHLTPEEKS AVTALWGKVN VDEVGGEALG RLLVVYPWTQ RFFESFGDLS</span><br><span class="line">TPDAVMGNPK VKAHGKKVLG AFSDGLAHLD NLKGTFATLS ELHCDKLHVD</span><br><span class="line">PENFRLLGNV LVCVLAHHFG KEFTPPVQAA YQKVVAGVAN ALAHKYH</span><br><span class="line"></span><br><span class="line">&gt;BRCA1_HUMAN </span><br><span class="line">MDLSALRVEEVQNVINAMQKILECPICLELIKEPVSTKCDHIFCKFCMLKLLNQKKGPSQ</span><br><span class="line">CPLCKNDITKRSLQESTRFSQLVEELLKIICAFQLDTGLEYANSYNFAKKENNSPEHLKD</span><br><span class="line">EVSIIQSMGYRNRAKRLLQSEPENPSLQETSLSVQLSNLGTVRTLRTKQRIQPQKTSVYI</span><br><span class="line">ELGSDSSEDTVNKATYCSVGDQELLQITPQGTRDEISLDSAKKAACEFSETDVTNTEHHQ</span><br><span class="line">PSNNDLNTTEKRAAERHPEKYQGSSVSNLHVEPCGTNTHASSLQHENSSLLLTKDRMNVE</span><br><span class="line">KAEFCNKSKQPGLARSQHNRWAGSKETCNDRRTPSTEKKVDLNADPLCERKEWNKQKLPC</span><br><span class="line">SENPRDTEDVPWITLNSSIQKVNEWFSRSDELLGSDDSHDGESESNAKVADVLDVLNEVD</span><br><span class="line">EYSGSSEKIDLLASDPHEALICKSERVHSKSVESNIEDKIFGKTYRKKASLPNLSHVTEN</span><br><span class="line">LIIGAFVTEPQIIQERPLTNKLKRKRRPTSGLHPEDFIKKADLAVQKTPEMINQGTNQTE</span><br><span class="line">QNGQVMNITNSGHENKTKGDSIQNEKNPNPIESLEKESAFKTKAEPISSSISNMELELNI</span><br><span class="line">HNSKAPKKNRLRRKSSTRHIHALELVVSRNLSPPNCTELQIDSCSSSEEIKKKKYNQMPV</span><br><span class="line">RHSRNLQLMEGKEPATGAKKSNKPNEQTSKRHDSDTFPELKLTNAPGSFTKCSNTSELKE</span><br><span class="line">FVNPSLPREEKEEKLETVKVSNNAEDPKDLMLSGERVLQTERSVESSSISLVPGTDYGTQ</span><br><span class="line">ESISLLEVSTLGKAKTEPNKCVSQCAAFENPKGLIHGCSKDNRNDTEGFKYPLGHEVNHS</span><br><span class="line">RETSIEMEESELDAQYLQNTFKVSKRQSFAPFSNPGNAEEECATFSAHSGSLKKQSPKVT</span><br><span class="line">FECEQKEENQGKNESNIKPVQTVNITAGFPVVGQKDKPVDNAKCSIKGGSRFCLSSQFRG</span><br><span class="line">NETGLITPNKHGLLQNPYRIPPLFPIKSFVKTKCKKNLLEENFEEHSMSPEREMGNENIP</span><br><span class="line">STVSTISRNNIRENVFKEASSSNINEVGSSTNEVGSSINEIGSSDENIQAELGRNRGPKL</span><br><span class="line">NAMLRLGVLQPEVYKQSLPGSNCKHPEIKKQEYEEVVQTVNTDFSPYLISDNLEQPMGSS</span><br><span class="line">HASQVCSETPDDLLDDGEIKEDTSFAENDIKESSAVFSKSVQKGELSRSPSPFTHTHLAQ</span><br><span class="line">GYRRGAKKLESSEENLSSEDEELPCFQHLLFGKVNNIPSQSTRHSTVATECLSKNTEENL</span><br><span class="line">LSLKNSLNDCSNQVILAKASQEHHLSEETKCSASLFSSQCSELEDLTANTNTQDPFLIGS</span><br><span class="line">SKQMRHQSESQGVGLSDKELVSDDEERGTGLEENNQEEQSMDSNLGEAASGCESETSVSE</span><br><span class="line">DCSGLSSQSDILTTQQRDTMQHNLIKLQQEMAELEAVLEQHGSQPSNSYPSIISDSSALE</span><br><span class="line">DLRNPEQSTSEKAVLTSQKSSEYPISQNPEGLSADKFEVSADSSTSKNKEPGVERSSPSK</span><br><span class="line">CPSLDDRWYMHSCSGSLQNRNYPSQEELIKVVDVEEQQLEESGPHDLTETSYLPRQDLEG</span><br><span class="line">TPYLESGISLFSDDPESDPSEDRAPESARVGNIPSSTSALKVPQLKVAESAQSPAAAHTT</span><br><span class="line">DTAGYNAMEESVSREKPELTASTERVNKRMSMVVSGLTPEEFMLVYKFARKHHITLTNLI</span><br><span class="line">TEETTHVVMKTDAEFVCERTLKYFLGIAGGKWVVSYFWVTQSIKERKMLNEHDFEVRGDV</span><br><span class="line">VNGRNHQGPKRARESQDRKIFRGLEICCYGPFTNMPTDQLEWMVQLCGASVVKELSSFTL</span><br><span class="line">GTGVHPIVVVQPDAWTEDNGFHAIGQMCEAPVVTREWVLDSVALYQCQELDTYLIPQIPH</span><br><span class="line">SHY</span><br><span class="line"></span><br><span class="line">&gt;21A [CPC](http://cpc.cbi.pku.edu.cn/)</span><br><span class="line">AAATAGTTGACCAAGTGTGGTGGCTCACGTAGTCCCAGCACTTTGGGAGGCTGAGGCAGGAGGATCACTTGAGCCCAGGAATTTGAGACCAGCTTGGGCAACATAGTGAGACCTCATCTCTTAAAAAAAAAAATTAGCTGGGTGTGGTAGTGCACACCTGTGGTCCCAGCTACTTTAGAGGCTGAGGTAGAGGATTGCTTGAGCCTGGGAAGTTGGGGCTGTAGTGAGCTTTGATTGCATCACTGCACTCCAGCCTGGGTGACAGAGCAAGACCCTGTCTCTAAAAAATTAAATAAATAATAAAAAAATTAAAAAGTAACTCCC</span><br></pre></td></tr></table></figure>
<p><img src="https://raw.githubusercontent.com/XIAPengfei1996/Shia/master/certificate.png" alt="certificate"></p>
<p><strong>从数学物理，到化学材料的科学理论突破，应用到工程界，引发信息编程等领域链式的技术革命，极大地推动了生物、医学等下游学科的发展。随着实验检测技术和试验设计的革新，人群研究逐渐能够摆脱伦理学的困境，内外暴露物质的测量方法也开始了急速地发展，直接从人类身上全面地获取有循证意义的数据成为可能。经典流行病学也将逆转被动研究黑箱问题的局面，使用生物信息学等方法，借助云储存、云计算分析海量组学数据，定量地研究全时空下暴露、分子、结局全水平的互作（interaction），彻底探清疾病机制，构建在人类医学生具有划时代意义的系统流行病学。而本就是限于伦理，“曲线救国”来研究人类疾病的动物实验，其现实意义也将大打折扣，甚至会在不久的将来只有博物馆的一席之地。</strong></p>
<blockquote>
<p>“Genetics Loads the Gun, Lifestyle Pulls the Trigger.”<br>—— Barbara O’Neill</p>
</blockquote>
<p><img src="http://care.diabetesjournals.org/content/diacare/36/5/1413/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" alt="Image of Interaction Network"><br>(Source: Franks PW, et al. Diabetes Care, 2013)</p>
<blockquote>
<p>从暴露组学发展到打开流行病学“黑箱子”的遗传组学、代谢组学等，系统流行病学可以多水平、多层次地考虑疾病的发生机制；除了机理研究之外，健康的社会决定因素和遗传易感性的影响因素研究，都进一步拓展了病因学研究的广度和深度。<br>——<a href="http://sph.pku.edu.cn/content/?2249.html" target="_blank" rel="noopener">李立明</a></p>
</blockquote>
<h2 id="华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师"><a href="#华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师" class="headerlink" title="华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师"></a>华中科技大学同济医学院公共卫生学院流行病学与卫生统计学系生物信息学方向导师</h2><ul>
<li><a href="http://gwxy.tjmu.edu.cn/info/1087/1796.htm" target="_blank" rel="noopener">龚静 博士</a></li>
<li><a href="http://chaolongwang.github.io" target="_blank" rel="noopener">王超龙 教授</a></li>
</ul>
<h2 id="参考书目"><a href="#参考书目" class="headerlink" title="参考书目"></a>参考书目</h2><ol>
<li>《生物信息学》, 李霞, 雷健波; 人民卫生出版社, 2015</li>
<li>《概率论与数理统计》, 盛骤, 谢式千, 潘承毅; 高等教育出版社, 2008</li>
</ol>
<h2 id="编程语言"><a href="#编程语言" class="headerlink" title="编程语言"></a>编程语言</h2><ul>
<li>Linux Shell / Bash<ul>
<li><a href="http://linux.vbird.org/linux_basic/" target="_blank" rel="noopener">鳥哥的Linux私房菜-基礎學習篇</a></li>
<li><a href="https://legacy.gitbook.com/book/wizardforcel/vbird-linux-basic-4e/details" target="_blank" rel="noopener">鸟哥的Linux私房菜：基础学习篇 第四版</a></li>
</ul>
</li>
<li>Git<ul>
<li><a href="http://iissnan.com/progit/" target="_blank" rel="noopener">《Pro Git》</a></li>
</ul>
</li>
<li>R<ul>
<li>《R语言实战》</li>
<li>《R语言与Bioconductor生物信息学应用》</li>
</ul>
</li>
<li>Perl<ul>
<li>《Perl语言入门》/《Learning Perl》传说中的“小骆驼”</li>
</ul>
</li>
<li>Python<ul>
<li>《Python基础教程》</li>
<li>《Python编程：从入门到实践》</li>
</ul>
</li>
<li>C</li>
</ul>

          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/lncRNA/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/lncRNA/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门"><a href="#转录组入门" class="headerlink" title="转录组入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门</a></h2><h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
      

  

  
  
  

  <article class="post post-type-normal" itemscope itemtype="http://schema.org/Article">
  
  
  
  <div class="post-block">
    <link itemprop="mainEntityOfPage" href="http://yoursite.com/2018/04/03/transcriptome/">

    <span hidden itemprop="author" itemscope itemtype="http://schema.org/Person">
      <meta itemprop="name" content="John Doe">
      <meta itemprop="description" content="">
      <meta itemprop="image" content="/images/avatar.gif">
    </span>

    <span hidden itemprop="publisher" itemscope itemtype="http://schema.org/Organization">
      <meta itemprop="name" content="Hexo">
    </span>

    
      <header class="post-header">

        
        
          <h1 class="post-title" itemprop="name headline">
                
                <a class="post-title-link" href="/2018/04/03/transcriptome/" itemprop="url">Untitled</a></h1>
        

        <div class="post-meta">
          <span class="post-time">
            
              <span class="post-meta-item-icon">
                <i class="fa fa-calendar-o"></i>
              </span>
              
                <span class="post-meta-item-text">Posted on</span>
              
              <time title="Post created" itemprop="dateCreated datePublished" datetime="2018-04-03T21:29:10+08:00">
                2018-04-03
              </time>
            

            

            
          </span>

          

          
            
          

          
          

          

          

          

        </div>
      </header>
    

    
    
    
    <div class="post-body" itemprop="articleBody">

      
      

      
        
          
            <h1 id="生物信息学实战"><a href="#生物信息学实战" class="headerlink" title="生物信息学实战"></a>生物信息学实战</h1><h2 id="转录组入门演习"><a href="#转录组入门演习" class="headerlink" title="转录组入门演习"></a><a href="https://xiapengfei1996.github.io/bioinfo/transcriptome" target="_blank" rel="noopener">转录组入门演习</a></h2><p><a href="https://mp.weixin.qq.com/s?__biz=MzAxMDkxODM1Ng==&amp;mid=2247484843&amp;idx=1&amp;sn=95370393e9d16e09e854adce78c94523&amp;chksm=9b484510ac3fcc064b1bfc19f5cc73a4fbc113a840a40c3a95632127c54f570d9d66550ebc22&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">任务清单</a>  <a href="https://mp.weixin.qq.com/s?__biz=MzAxMDkxODM1Ng==&amp;mid=2247485440&amp;idx=1&amp;sn=392c8649e334af99ac60e903aaec01e9&amp;chksm=9b4848bbac3fc1ad02ac2f11fd9f4440fb13cffc063d19da56fe34317d0621a4972ec26a41d1&amp;mpshare=1&amp;scene=23&amp;srcid=04136Qg8PRlWmG5bwOcYtjwq%23rd" target="_blank" rel="noopener">参考流程</a></p>
<h3 id="计算机资源的准备"><a href="#计算机资源的准备" class="headerlink" title="计算机资源的准备"></a>计算机资源的准备</h3><ul>
<li><p>系统环境及硬件配置：</p>
<ul>
<li>macOS High Sierra 10.13.4</li>
<li>MacBool Air (13-inch, Early 2015)</li>
<li>处理器   1.6GHz intel Core i5</li>
<li>内存  4GB 1600MHz DDR3</li>
<li>存储  128G</li>
<li><code>uname -a</code>  x86_64</li>
</ul>
</li>
<li><p>安装Homebrew包管理，要求预先安装Xcode；<br>（App Store中Xcode要求macOS High Sierra 10.13.2以上）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"</span><br></pre></td></tr></table></figure>
</li>
<li><p>用homebrew安装wget；（也可在<a href="rudix.org/packages/wget.html">rudix网站</a>上下载wget安装包）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">brew install wget</span><br></pre></td></tr></table></figure>
</li>
<li><p><a href="https://zhuanlan.zhihu.com/p/25085567" target="_blank" rel="noopener">wget下载miniconda</a>；<br>此处注意网上教程多在Linux或Windows虚拟Ubantu系统下操作，要注意下载MacOSX版文件！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">wget https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh</span><br></pre></td></tr></table></figure>
</li>
<li><p>bash安装miniconda；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">bash Miniconda3-latest-MacOSX-x86_64.sh</span><br></pre></td></tr></table></figure>
</li>
<li><p>输入yes接受条款；</p>
</li>
<li><p>回车确认安装位置；</p>
</li>
<li><p>输入yes确定PATH；</p>
</li>
<li><p>source完成最后安装；<br>此处注意MacOSX系统中不是<code>.bashrc</code>，应根据反馈结果修改为<code>.bash_profile</code>！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">source ~/.bash_profile</span><br></pre></td></tr></table></figure>
</li>
<li><p>查看是否安装成功；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda</span><br></pre></td></tr></table></figure>
</li>
<li><p>更新miniconda；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda update conda</span><br></pre></td></tr></table></figure>
</li>
<li><p>添加channels,即为conda增加软件源；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">conda config - add channels r</span><br><span class="line">conda config - add channels bioconda</span><br><span class="line">conda config - add channels defaults</span><br><span class="line">conda config – add channels conda-forge</span><br></pre></td></tr></table></figure>
</li>
<li><p>查看添加的频道；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda config – get channels</span><br></pre></td></tr></table></figure>
</li>
<li><p>添加清华源镜像；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">conda config - add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/</span><br><span class="line">conda config - add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/</span><br><span class="line">conda config - add channels https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/msys2/</span><br></pre></td></tr></table></figure>
</li>
<li><p>显示已添加的源镜像地址；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">conda config --set show_channel_urls yes</span><br></pre></td></tr></table></figure>
</li>
<li><p>安装sratoolkit,fastqc,hisats,samtools,htseq-count,R,Rstudio软件；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">conda install -c bioconda sra-tools</span><br><span class="line">conda install -c bioconda fastqc</span><br><span class="line">conda install -c bioconda hisat2</span><br><span class="line">conda install -c bioconda samtools</span><br><span class="line">conda install -c bioconda htseq</span><br><span class="line"><span class="meta">#</span> 本机已安装过R和RStudio,故未重复下载</span><br><span class="line"><span class="meta">#</span> conda install r</span><br><span class="line"><span class="meta">#</span> conda install rstudio</span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="读文章拿到测序数据"><a href="#读文章拿到测序数据" class="headerlink" title="读文章拿到测序数据"></a>读文章拿到测序数据</h3><ul>
<li><p>检索实战学习示例文章；<br><a href="https://www.nature.com/articles/ncomms13347" target="_blank" rel="noopener">AKAP95 regulates splicing through scaffolding RNAs and RNA processing factors. Nat Commun 2016 Nov 8;7:13347. PMID: 27824034</a></p>
</li>
<li><p>阅读文章Methods部分，尤Bioinformatic analyses和Data availability部分；</p>
<blockquote>
<p>Bioinformatic analyses<br>We obtained 35–55 million of paired 51 bp reads for each sample. The investigator was blinded to sample identities when performing bioinformatic analyses. All the reads were mapped to the <strong>human reference genome (GRCh37/hg19)</strong> using <strong>TopHat (v2.0.13)</strong> with a <strong>gene transfer file (GTF version GRCh37.70)</strong>. <strong>Low quality mapped reads (MQ&gt;30)</strong> were removed from the analysis. The <strong>mean insert sizes and the s.d.’s</strong> were calculated using <a href="http://broadinstitute.github.io/picard/" target="_blank" rel="noopener"><strong>Picard-tools (v1.126)</strong></a>.<br>For mRNA-Seq, <strong>read count tables</strong> were generated using <strong>HTSeq (v0.6.0)53</strong> and <strong>deferential expression (DE) analysis of genes</strong> were performed using <strong>DESeq (v3.0)54</strong>. <strong>Deferential exon usage</strong> was performed using <strong>DEXSeq (v3.1)29</strong>. Two independent biological repeats for control and KD were included in both DEseq and DEXseq. <em><strong>For 293 cells, the mRNA-seq results of the control samples include (1) those from the doxycycline-treated parental Flp-In T-REx 293 cells by us and (2) those from the doxycycline-treated control Flp-In T-REx 293 cells performed by another group unrelated to us (sample GSM1095127 in GSE44976)31, and the two AKAP95 KD samples were from the two different clones (miR#8 and miR#12) of Flp-In T-REx 293 cells stably expressing AKAP95 microRNAs after doxycycline treatment.</strong></em> For mouse ES cells, the two biological repeats for both control and shRNA#1-mediated Akap95 KD were independently generated by two different persons in our lab. The <strong>read per million normalized BigWig files</strong> were generated using <strong>BEDTools (v2.17.0)55</strong> and <strong>bedGraphToBigWig tool (v4)</strong>. All the <strong>downstream statistical analyses</strong> and generating <strong>plots</strong> were performed in <a href="http://www.r-project.org/" target="_blank" rel="noopener"><strong>R (v3.1.1)</strong></a>. <strong>Gene ontology analysis</strong> was performed at <a href="http://david.ncifcrf.gov/" target="_blank" rel="noopener"><strong>DAVID</strong></a>.<br>For RIP-Seq, mapped files were passed to Model-based Analysis of ChIP-Seq (MACS) (v1.4.2 20120305)56 for peak calling and <a href="http://homer.salk.edu/homer/" target="_blank" rel="noopener">HOMER (vv3.12, 6-8-2012)</a>57 was used for Motif finding. Bedgraph files generated by MACS were then normalized based on read per million and converted to BigWig files using bedGraphToBigWig. The input RNA-seq files were not subject to MACS. The genomic regions for annotation (intergenic regions, exons, UTRs, distal and proximal introns) were extracted from the GTF file used above. Peak annotations and coverage calculations for these genomic regions were performed using Bedtools and SAMtools (v0.1.18)58, respectively. Profile plots were generated by ngs.plot (v2.47)59.</p>
</blockquote>
</li>
</ul>
<blockquote>
<p>Data availability<br>The RIP-seq an RNA-seq data have been deposited in the Gene Expression Omnibus database, with accession code <strong>GSE81916</strong>. All other data is available from the author upon reasonable request.</p>
</blockquote>
<ul>
<li>在<a href="https://www.ncbi.nlm.nih.gov/geo/" target="_blank" rel="noopener">GEO</a>上查找<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81916" target="_blank" rel="noopener"><strong>GSE81916</strong></a>,查找得其对应的SRA号为<strong>SRP075747</strong>（以及BioProject号<strong>PRJNA323422</strong>）；</li>
</ul>
<blockquote>
<p>Overall design<br>Samples 1-8 are RNA-immunoprecipitation (RIP)-seq to determine AKAP95 binding to the transcriptome. <strong>Samples 9-15 are mRNA-seq</strong> to determine effect of AKAP95 knockdown in human 293 cells (9-11) or mouse ES cells (12-15).</p>
</blockquote>
<ul>
<li><p>阅读网页Overall design部分介绍得知，属于RNA-seq数据的是第9-15个样品；</p>
</li>
<li><p>在<a href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=search_obj" target="_blank" rel="noopener">SRA</a>上查找<a href="https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP075747&amp;go=go" target="_blank" rel="noopener"><strong>SRP075747</strong></a>（或<a href="https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=PRJNA323422" target="_blank" rel="noopener"><strong>PRJNA323422</strong></a>）,查找得其第9-15个样品，即<strong>GSM2177723</strong>到<strong>GSM2177729</strong>对应的SRA号为<strong>SRR3589956</strong>到<strong>SRR3589962</strong>；<br><em>注意理解GEO/SRA数据库的数据存放形式</em></p>
</li>
<li><p>安装axel来下载数据比wget及prefetch更快；（也可使用<a href="http://www.bio-info-trainee.com/bioconductor_China/software/GEOquery.html" target="_blank" rel="noopener">GEOquery</a>下载）<br>（Aspera只有Windows系统下的client）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">brew install axel</span><br></pre></td></tr></table></figure>
</li>
<li><p>循环axel语句实现数据下载；<br>（<strong>2200+KB/s，用时约两小时</strong>）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">for i in &#123;56..62&#125;  # &#123;56..62&#125;与`seq 57 62`等价</span><br><span class="line">do</span><br><span class="line">  axel ftp://ftp-trace.ncbi.nih.gov/sra/sra-instant/reads/ByStudy/sra/SRP/SRP075/SRP075747/SRR35899$i/SRR35899$i.sra</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="了解fastq测序数据"><a href="#了解fastq测序数据" class="headerlink" title="了解fastq测序数据"></a>了解fastq测序数据</h3><ul>
<li><p>循环sratoolkit（sra-tools）中fastq-dump语句把sra文件转换为fastq格式；<br>（<strong>用时约三个半小时</strong>）<br>注意这一步不显示百分数进度条！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">fastq-dump -h # 查看帮助文件</span><br><span class="line">for i in &#123;56..62&#125;</span><br><span class="line">do</span><br><span class="line">  fastq-dump --gzip --split-3 -O /Users/xiapengfei/ -A SRR35899$&#123;i&#125;.sra</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
<li><p>循环fastqc语句测试测序文件的质量；(<a href="http://fbb84b26.wiz03.com/share/s/3XK4IC0cm4CL22pU-r1HPcQQ1iRTvV2GwkwL2AaxYi2fXHP7" target="_blank" rel="noopener">MultiQC可以批量显示质控结果</a>）<br>（<strong>4线程，用时约25分钟</strong>）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">for i in &#123;56..62&#125;</span><br><span class="line">do</span><br><span class="line">  fastqc -t 4 -O /Users/xiapengfei/ SRR35899$i.sra_1.fastq.gz SRR35899$i.sra_2.fastq.gz</span><br><span class="line">done  # fastqc *.fastq在当前目录中无其他同类文件时等价，其他循环或可同理</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p>_<a href="https://github.com/XIAPengfei1996/Shia/tree/master/bioinfo/fastqc_html" target="_blank" rel="noopener">理解测序reads，GC含量，质量值，接头，index，fastqc的全部报告</a>_</p>
<p><a href="http://www.bio-info-trainee.com/2275.html" target="_blank" rel="noopener"><strong>经验教训</strong></a></p>
<h3 id="了解参考基因组及基因注释"><a href="#了解参考基因组及基因注释" class="headerlink" title="了解参考基因组及基因注释"></a>了解参考基因组及基因注释</h3><ul>
<li>在<a href="http://genome.ucsc.edu/index.html" target="_blank" rel="noopener">UCSC</a>中打开Downloads中Genome Data页面，点击Human选择Feb. 2009 (GRCh37/hg19)栏下Full data set，下拉找到chromFa.tar.gz文件；</li>
</ul>
<blockquote>
<p>chromFa.tar.gz - The assembly sequence in one file per chromosome.<br>    Repeats from RepeatMasker and Tandem Repeats Finder (with period<br>    of 12 or less) are shown in lower case; non-repeating sequence is<br>    shown in upper case.</p>
</blockquote>
<ul>
<li><p>axel命令下载hg19参考基因组数据；<br>（<strong>400+KB/s，用时约35分钟</strong>）</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">axel http://hgdownload.soe.ucsc.edu/goldenPath/hg19/bigZips/chromFa.tar.gz</span><br><span class="line">tar zxvf chromFa.tar.gz  # 解压该文件</span><br><span class="line">cat *.fa &gt; hg19.fa  # 将解压文件合并为一个文件</span><br><span class="line">rm chr*.fa  # 删除无关数据</span><br></pre></td></tr></table></figure>
</li>
<li><p>从<a href="http://www.gencodegenes.org" target="_blank" rel="noopener">gencode数据库</a>中打开Data中Human栏内<a href="https://www.gencodegenes.org/releases/grch37_mapped_releases.html" target="_blank" rel="noopener">GRCh37-mapped Releases</a>，点击GENCODE release栏下最新版的数据，下载基因注释文件；<br>（<strong>用时5分钟</strong>)</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">axel ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/GRCh37_mapping/gencode.v28lift37.annotation.gtf.gz  # GTF格式主要是用来描述基因的注释</span><br><span class="line">axel ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_28/GRCh37_mapping/gencode.v28lift37.annotation.gff3.gz  # GFF文件是一种用来描述基因组特征的文件，现在我们所使用的大部分都是第三版（GFF3）</span><br><span class="line"><span class="meta">#</span> gzip -d解压并删除压缩包</span><br><span class="line">gzip -d gencode.v28lift37.annotation.gtf.gz</span><br><span class="line">gzip -d gencode.v28lift37.annotation.gff3.gz</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>- 用IGV去查看感兴趣的基因的结构；</em></p>
<ul>
<li>下载安装<a href="https://java.com/en/download/mac_download.jsp" target="_blank" rel="noopener">JAVA 8</a>；</li>
<li>下载<a href="http://software.broadinstitute.org/software/igv/download" target="_blank" rel="noopener">igv</a>直接打开；</li>
<li>在菜单栏Genomes中选择Load Genome from File…导入hg19.fa文件；</li>
<li>在菜单栏Tools中打开Run igvtools…界面，在Command栏中选择Sort，在Input File栏中导入gtf文件后运行；</li>
<li>在菜单栏File中选择Load from File…导入sorted.gtf文件并index；</li>
<li>在工具栏输入查找的基因名词（如：TP53）或染色体定位（如chr17:7,569,720-7,592,868)查看<a href="https://github.com/XIAPengfei1996/Shia/tree/master/bioinfo/igv" target="_blank" rel="noopener">可视化结构</a>；</li>
</ul>
<h3 id="序列比对"><a href="#序列比对" class="headerlink" title="序列比对"></a>序列比对</h3><ul>
<li>从<a href="http://ccb.jhu.edu/software/hisat2/index.shtml" target="_blank" rel="noopener">HISAT2</a>的主页下载index文件；<br>（<strong>400+KB/s，用时约两个半+小时</strong>）<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">axel ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/data/hg19.tar.gz  # 下载人类的索引</span><br><span class="line">axel ftp://ftp.ccb.jhu.edu/pub/infphilo/hisat2/data/mm10.tar.gz  # 下载小鼠的索引</span><br><span class="line">tar -zxvf *.tar.gz  # 解压压缩包</span><br><span class="line">rm -rf *.tar.gz  # 删除压缩包</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p>这次分析是人类mRNA-Seq测序的结果，但是这里其实只下载了3个人类的sra文件。一般而言RNA-Seq数据分析都至少要有2个重复，所以必须要有4个sra文件才行。实际上前面原文Bioinformatic analyses部分有说该研究control组中的一组数据用的是其他课题组的：</p>
<blockquote>
<p><em><strong>For 293 cells, the mRNA-seq results of the control samples include (1) those from the doxycycline-treated parental Flp-In T-REx 293 cells by us and (2) those from the doxycycline-treated control Flp-In T-REx 293 cells performed by another group unrelated to us (sample GSM1095127 in GSE44976)31, and the two AKAP95 KD samples were from the two different clones (miR#8 and miR#12) of Flp-In T-REx 293 cells stably expressing AKAP95 microRNAs after doxycycline treatment.</strong></em></p>
</blockquote>
<p>不同来源的RNA-Seq数据的构建文库不一样,结果之间比较需要考虑<strong>批次效应（batch effect）</strong> 的影响<br>IVT-seq reveals extreme bias in RNA-sequencing 证明了不同文库的RNA-Seq结果会存在很大差异</p>
<ul>
<li><p>把fastq格式的reads比对上去得到sam文件(仅小鼠，下同)；<br>注意这一步也不显示百分数进度条！</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br></pre></td><td class="code"><pre><span class="line"><span class="meta">#</span> 比对人类的索引 9:55</span><br><span class="line"><span class="meta">#</span> for i in &#123;56..58&#125;</span><br><span class="line"><span class="meta">#</span> do</span><br><span class="line"><span class="meta">#</span>   hisat2 -t -x /Users/xiapengfei/hg19/genome -1 SRR35899$&#123;i&#125;.sra_1.fastq.gz -2 SRR35899$&#123;i&#125;.sra_2.fastq.gz -S SRR35899$&#123;i&#125;.sam</span><br><span class="line"><span class="meta">#</span> done</span><br><span class="line"></span><br><span class="line"><span class="meta">#</span> 比对小鼠的索引</span><br><span class="line">for i in &#123;59..62&#125;</span><br><span class="line">do</span><br><span class="line">  hisat2 -t -x /Users/xiapengfei/hg19/genome -1 SRR35899$&#123;i&#125;.sra_1.fastq.gz -2 SRR35899$&#123;i&#125;.sra_2.fastq.gz -S SRR35899$&#123;i&#125;.sam</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
<li><p>用samtools把sam文件转为bam文件，并且排序(注意N和P两种排序区别)索引好；</p>
<figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line">for i in &#123;59..62&#125;</span><br><span class="line">do </span><br><span class="line">  samtools view -S SRR35899$&#123;i&#125;.sam -b &gt; SRR35899$&#123;i&#125;.bam  # 将比对后的sam格式文件转换成bam文件</span><br><span class="line">  samtools sort SRR35899$&#123;i&#125;.bam -o SRR35899$&#123;i&#125;.sorted.bam  # 将所有的bam文件进行排序</span><br><span class="line">  samtools index SRR35899$&#123;i&#125;_sorted.bam # 将所有的排序文件建立索引</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>samtools sort默认按照染色体位置进行排序，-n参数根据read名进行排序。-t参数根据TAG进行排序</em></p>
<ul>
<li>对bam文件进行质控；<br>可以使用<a href="http://rseqc.sourceforge.net/" target="_blank" rel="noopener">RSeQC</a><br><em>RSeQC的安装，需要预先安装R，Python2.7，gcc，<a href="http://www.jianshu.com/p/14fd4de54402" target="_blank" rel="noopener">numpy</a></em><figure class="highlight python"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">pip install RSeQC</span><br><span class="line"><span class="keyword">for</span> i <span class="keyword">in</span> &#123;<span class="number">56.</span><span class="number">.62</span>&#125;</span><br><span class="line">do</span><br><span class="line">  bam_stat.py -i SRR35899$&#123;i&#125;_sorted.bam</span><br><span class="line">done</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>另有<a href="http://broadinstitute.github.io/picard/" target="_blank" rel="noopener">Picard</a></em><br><figure class="highlight java"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br></pre></td><td class="code"><pre><span class="line">conda install ucsc-gtftogenepred</span><br><span class="line">gtfToGenePred -genePredExt -ignoreGroupsWithoutExons -geneNameAsName2 gencode.v28lift37.annotation.gtf gencode.v28lift37.annotation.gpd</span><br><span class="line">axel http:<span class="comment">//hgdownload.soe.ucsc.edu/goldenPath/hg19/database/refFlat.txt.gz</span></span><br><span class="line">gzip -d refFlat.txt.gz</span><br><span class="line">java -jar ~/biosoft/picard_2.8.0/picard.jar CollectRnaSeqMetrics I= Homo_sapiens_Control_293_cell_sorted.bam O=Homo_sapiens_Control_293_cell_RNA_Metrics REF_FLAT= ~/rna_seq/data/reference/genome/hg19/refFlat.txt STRAND_SPECIFICITY=NONE</span><br></pre></td></tr></table></figure></p>
<p><em>或<a href="http://qualimap.bioinfo.cipf.es/" target="_blank" rel="noopener">Qualimap</a>等工具</em></p>
<ul>
<li>IGV载入参考序列，注释和BAM文件查看略；</li>
</ul>
<h3 id="reads计数"><a href="#reads计数" class="headerlink" title="reads计数"></a>reads计数</h3><p>用htseq对比得到的bam文件进行count计数；<br><figure class="highlight shell"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">for i in $(seq 56 58)</span><br><span class="line">do </span><br><span class="line">  htseq-count -f bam -r pos -s no SRR35899$&#123;i&#125;.bam gencode.vM10.annotation.gtf SRR35899$&#123;i&#125;.count</span><br><span class="line">done</span><br></pre></td></tr></table></figure></p>
<ul>
<li>将count文件合并为表达矩阵；（59和61是control， 60和62是repeat）<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br></pre></td><td class="code"><pre><span class="line">options(stringsAsFactors = <span class="literal">FALSE</span>)</span><br><span class="line"><span class="comment"># 导入表格数据</span></span><br><span class="line">control1 &lt;- read.table(<span class="string">"SRR3589959.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"control"</span>))</span><br><span class="line">repeat1 &lt;- read.table(<span class="string">"SRR3589960.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"akap951"</span>))</span><br><span class="line">control1 &lt;- read.table(<span class="string">"SRR3589961.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"control"</span>))</span><br><span class="line">repeat2 &lt;- read.table(<span class="string">"SRR3589962.count"</span>, sep=<span class="string">"\t"</span>, col.names = c(<span class="string">"gene_id"</span>,<span class="string">"akap952"</span>))</span><br><span class="line"><span class="comment"># 按照gene_id合并</span></span><br><span class="line">raw_count &lt;- merge(merge(control1, control2, by=<span class="string">"gene_id"</span>), merge(repeat1, repeat2, by=<span class="string">"gene_id"</span>))</span><br><span class="line"><span class="comment"># 过滤删去5行</span></span><br><span class="line">raw_count_filt &lt;- raw_count[-<span class="number">48823</span>:-<span class="number">48825</span>, ]</span><br><span class="line">raw_count_filter &lt;- raw_count_filt[-<span class="number">1</span>:-<span class="number">2</span>, ]</span><br><span class="line"><span class="comment"># 因为EBI数据库无法直接搜索小数位，因此需要替换为整数形式</span></span><br><span class="line">ENSEMBL &lt;- gsub(<span class="string">"\\.\\d*"</span>, <span class="string">""</span>, raw_count_filter$gene_id)</span><br><span class="line"><span class="comment"># 将ENSEMBL重新添加到矩阵中</span></span><br><span class="line">row.names(raw_count_filter) &lt;- ENSEMBL</span><br><span class="line"><span class="comment"># 在EBI或UniProt数据库找到AKAP95的id，并从矩阵中找到访问，并赋值给AKAP95变量</span></span><br><span class="line">&gt; AKAP95 &lt;- raw_count_filt[rownames(raw_count_filt)==<span class="string">"ENSG00000105127"</span>,]</span><br><span class="line"><span class="comment"># 查看AKAP95基因表达情况，结果明显比对照组高</span></span><br><span class="line">&gt; AKAP95</span><br><span class="line"><span class="comment"># 查看总体情况</span></span><br><span class="line">summary(raw_count_filter)</span><br></pre></td></tr></table></figure>
</li>
</ul>
<p><em>另可使用perl</em><br><figure class="highlight perl"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br></pre></td><td class="code"><pre><span class="line">perl -lne <span class="string">'if($ARGV=~/Homo_sapiens_(.*)_sorted.count/)&#123;print "$1\t$_"&#125;'</span> *|<span class="keyword">grep</span> -v  Homo_sapiens&gt;hg.count</span><br><span class="line"><span class="comment"># 先把所有文件进行合并</span></span><br><span class="line">setwd(<span class="string">"~/rna_seq/work/matrix"</span>)</span><br><span class="line">hg &lt;- read.csv(file = <span class="string">"hg.count"</span>,header = F,sep = <span class="string">"\t"</span>)</span><br><span class="line">colnames(hg) &lt;- c(<span class="string">'sample'</span>,<span class="string">'gene'</span>,<span class="string">'count'</span>)</span><br><span class="line">library(reshape2)</span><br><span class="line">reads &lt;- dcast(hg,formula = gene ~ sample)</span><br><span class="line">write.table(reads,file = <span class="string">"hg_join.count"</span>,sep = <span class="string">"\t"</span>,quote = FALSE,row.names = FALSE)</span><br><span class="line">summary(reads)</span><br><span class="line">sample_mean &lt;- group_by(hg,sample)%&gt;% summarize(mean=mean(count))</span><br><span class="line">gene_mean &lt;- group_by(hg,gene)%&gt;% summarize(mean=mean(count)) </span><br><span class="line">tmp &lt;- full_join(hg,gene_mean,by=<span class="string">'gene'</span>)</span><br></pre></td></tr></table></figure></p>
<h3 id="差异基因分析"><a href="#差异基因分析" class="headerlink" title="差异基因分析"></a>差异基因分析</h3><p>参考：<br><a href="https://zhuanlan.zhihu.com/p/30350531" target="_blank" rel="noopener">青山屋主</a>  <a href="https://mp.weixin.qq.com/s/1mda7NpOxShbOxZKSJar7Q" target="_blank" rel="noopener">徐洲更</a>  <a href="https://mp.weixin.qq.com/s?__biz=MzIwNTEwMTUyOQ==&amp;mid=2649694709&amp;idx=1&amp;sn=40b354fef74d3adf5aac7b8745303af2&amp;chksm=8f2d8143b85a0855404621656c5946acc4919cc61a65719c450a461ecbdeb1ff56c72fe798bb&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">PANDA姐</a>/<a href="https://mp.weixin.qq.com/s?__biz=MzIwNTEwMTUyOQ==&amp;mid=2649694735&amp;idx=1&amp;sn=5ad87d24643c08ab945288e086da4359&amp;chksm=8f2d82b9b85a0baf92cc6af6f647c7e89710138a3f95e5e0a939894f21034edc57d5f7a30f52&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">沈梦圆</a></p>
<ul>
<li><p>在R中安装DESeq2；（也可使用edgeR，limma等）</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br></pre></td><td class="code"><pre><span class="line">rm(list=ls())</span><br><span class="line"><span class="keyword">source</span>(<span class="string">"https://bioconductor.org/biocLite.R"</span>) <span class="comment"># 载入安装工具</span></span><br><span class="line">biocLite(<span class="string">"DESeq2"</span>)  <span class="comment"># 安装包</span></span><br><span class="line"><span class="keyword">library</span>(<span class="string">"DESeq2"</span>) <span class="comment"># 加载包，测试是否安装成功</span></span><br></pre></td></tr></table></figure>
</li>
<li><p>正式构建dds矩阵；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br></pre></td><td class="code"><pre><span class="line">countData &lt;- raw_count_filter</span><br><span class="line">condition &lt;- factor(c(<span class="string">"control"</span>,<span class="string">"control"</span>,<span class="string">"KD"</span>,<span class="string">"KD"</span>))</span><br><span class="line">colData &lt;- data.frame(row.names=colnames(countData), condition)</span><br><span class="line">dds &lt;- DESeqDataSetFromMatrix(countData, DataFrame(condition), design= ~ condition )</span><br><span class="line">head(dds)  <span class="comment">#查看一下构建好的矩阵</span></span><br><span class="line">dds &lt;- DESeq(dds)   <span class="comment">#对原始dds进行normalize</span></span><br><span class="line"><span class="comment"># 查看结果的名称，本次实验中是 "Intercept"，"condition_akap95_vs_control"</span></span><br><span class="line">resultsNames(dds2)</span><br><span class="line"><span class="comment"># 将结果用results()函数来获取，赋值给res变量</span></span><br><span class="line">res &lt;- results(dds2)</span><br><span class="line"><span class="comment"># summary一下，看一下结果的概要信息</span></span><br><span class="line">summary(res)</span><br></pre></td></tr></table></figure>
</li>
<li><p>提取差异分析结果；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line"><span class="comment"># 获取padj（p值经过多重校验校正后的值）小于0.05，表达倍数取以2为对数后大于1或者小于-1的差异表达基因。</span></span><br><span class="line">&gt; table(res$padj&lt;<span class="number">0.05</span>) <span class="comment">#取P值小于0.05的结果</span></span><br><span class="line">&gt; res &lt;- res[order(res$padj),]</span><br><span class="line">&gt; diff_gene_deseq2 &lt;-subset(res,padj &lt; <span class="number">0.05</span> &amp; (log2FoldChange &gt; <span class="number">1</span> | log2FoldChange &lt; -<span class="number">1</span>))</span><br><span class="line">&gt; diff_gene_deseq2 &lt;- row.names(diff_gene_deseq2)</span><br><span class="line">&gt; resdata &lt;-  merge(as.data.frame(res),as.data.frame(counts(dds2,normalize=<span class="literal">TRUE</span>)),by=<span class="string">"row.names"</span>,sort=<span class="literal">FALSE</span>)</span><br><span class="line"><span class="comment"># 得到csv格式的差异表达分析结果</span></span><br><span class="line">&gt; write.csv(resdata,file= <span class="string">"control_vs_akap95.cvs"</span>,row.names = <span class="literal">F</span>)</span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="差异基因结果注释-富集分析"><a href="#差异基因结果注释-富集分析" class="headerlink" title="差异基因结果注释/富集分析"></a>差异基因结果注释/富集分析</h3><p>参考：<br><a href="https://mp.weixin.qq.com/s/p-n5jq5Rx2TqDBStS2nzoQ" target="_blank" rel="noopener">徐洲更</a>  <a href="https://www.jianshu.com/p/95c0472f5a8e" target="_blank" rel="noopener">宇天</a>  <a href="https://mp.weixin.qq.com/s?__biz=MzIwNTEwMTUyOQ==&amp;mid=2649694917&amp;idx=1&amp;sn=a318f8cf98f306d46963986011c73600&amp;chksm=8f2d8273b85a0b65270a986f7bb28e8efa7fd15309505a5a026bbedaafaff7e30e4d4f13dd02&amp;scene=21#wechat_redirect" target="_blank" rel="noopener">沈梦圆</a></p>
<ul>
<li><p>根据padj &lt; 0.05 且Log2FoldChange的绝对值大于1的标准筛选显著差异表达基因集；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">deseq2.sig &lt;- subset(res, padj &lt; <span class="number">0.05</span> &amp; abs(log2FoldChange) &gt; <span class="number">1</span>)</span><br></pre></td></tr></table></figure>
</li>
<li><p>安装下载注释数据包；（rog.HS.eg.db)</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">source</span>(<span class="string">"https://bioconductor.org/biocLite.R"</span>)</span><br><span class="line">biocLite(<span class="string">"clusterProfiler"</span>)</span><br><span class="line">biocLite（<span class="string">"AnnotationHub"</span>)</span><br><span class="line"><span class="keyword">library</span>(AnnotationHub)</span><br><span class="line">ah &lt;- AnnotationHub()</span><br><span class="line">org.hs &lt;- ah[[<span class="string">'AH53766'</span>]]</span><br></pre></td></tr></table></figure>
</li>
<li><p>GO富集和GESA；<br><em>Y叔的clusterProfiler简单好用，结果正确，没有必要专门导出数据用GSEA软件做超几何分布检验分析了</em></p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br></pre></td><td class="code"><pre><span class="line">ego &lt;- enrichGO(</span><br><span class="line">  gene = row.names(deseq2.sig),</span><br><span class="line">  OrgDb = org.hs,</span><br><span class="line">  keytype = <span class="string">"ENSEMBL"</span>,</span><br><span class="line">  ont = <span class="string">"MF"</span></span><br><span class="line">)</span><br><span class="line">dotplot(ego,font.size=<span class="number">5</span>)  <span class="comment"># 气泡图</span></span><br><span class="line">enrichMap(ego, vertex.label.cex=<span class="number">1.2</span>, layout=igraph::layout.kamada.kawai)</span><br><span class="line">plotGOgraph(ego)  <span class="comment"># 网络图</span></span><br><span class="line">gseaplot(gsemf, geneSetID=<span class="string">"GO:0004871"</span>)  <span class="comment"># 画GSEA标志性图</span></span><br></pre></td></tr></table></figure>
</li>
<li><p>KEGG富集分析；</p>
<figure class="highlight r"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br></pre></td><td class="code"><pre><span class="line"><span class="keyword">library</span>(clusterProfiler)</span><br><span class="line">gene_list &lt;- mapIds(org.hs, keys = row.names(deseq2.sig),</span><br><span class="line">  column = <span class="string">"ENTREZID"</span>, keytype = <span class="string">"ENSEMBL"</span> )</span><br><span class="line">kk &lt;- enrichKEGG(gene_list, organism=<span class="string">"hsa"</span>,</span><br><span class="line">  keyType = <span class="string">"ncbi-geneid"</span>,</span><br><span class="line">  pvalueCutoff=<span class="number">0.05</span>, pAdjustMethod=<span class="string">"BH"</span>,</span><br><span class="line">  qvalueCutoff=<span class="number">0.1</span>)</span><br><span class="line">head(summary(kk))</span><br><span class="line">dotplot(kk)  <span class="comment"># 气泡图</span></span><br></pre></td></tr></table></figure>
</li>
</ul>
<h3 id="致谢"><a href="#致谢" class="headerlink" title="致谢"></a>致谢</h3><p>感谢曾健明在其建立的生信技能树平台发起转录组入门实战项目；<br>感谢徐洲更、沈梦圆、青山屋主等在各平台上热心分享经验以资参考；<br>感谢所有论文作者和数据共享平台开放数据获取途径；<br>获益良多，相见恨晚！</p>
<h3 id="参考文献"><a href="#参考文献" class="headerlink" title="参考文献"></a>参考文献</h3><p>《RNA-seq数据分析实用方法》<br>（《RNA-seq Data Analysis A Practical Approach》）</p>
<h2 id="ChIP-seq基础入门"><a href="#ChIP-seq基础入门" class="headerlink" title="ChIP-seq基础入门"></a><a href="https://xiapengfei1996.github.io/bioinfo/ChIP-seq" target="_blank" rel="noopener">ChIP-seq基础入门</a></h2><h2 id="450K甲基化芯片数据处理"><a href="#450K甲基化芯片数据处理" class="headerlink" title="450K甲基化芯片数据处理"></a><a href="https://xiapengfei1996.github.io/bioinfo/Methylation450K_BeadChip" target="_blank" rel="noopener">450K甲基化芯片数据处理</a></h2><h2 id="lncRNA数据分析"><a href="#lncRNA数据分析" class="headerlink" title="lncRNA数据分析"></a><a href="https://xiapengfei1996.github.io/bioinfo/lncRNA" target="_blank" rel="noopener">lncRNA数据分析</a></h2>
          
        
      
    </div>
    
    
    

    

    

    

    <footer class="post-footer">
      

      

      

      
      
        <div class="post-eof"></div>
      
    </footer>
  </div>
  
  
  
  </article>


    
  </section>

  


          </div>
          


          

        </div>
        
          
  
  <div class="sidebar-toggle">
    <div class="sidebar-toggle-line-wrap">
      <span class="sidebar-toggle-line sidebar-toggle-line-first"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-middle"></span>
      <span class="sidebar-toggle-line sidebar-toggle-line-last"></span>
    </div>
  </div>

  <aside id="sidebar" class="sidebar">
    
    <div class="sidebar-inner">

      

      

      <section class="site-overview-wrap sidebar-panel sidebar-panel-active">
        <div class="site-overview">
          <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
            
              <p class="site-author-name" itemprop="name">John Doe</p>
              <p class="site-description motion-element" itemprop="description"></p>
          </div>

          <nav class="site-state motion-element">

            
              <div class="site-state-item site-state-posts">
              
                <a href="/archives/">
              
                  <span class="site-state-item-count">8</span>
                  <span class="site-state-item-name">posts</span>
                </a>
              </div>
            

            

            

          </nav>

          

          

          
          

          
          

          

        </div>
      </section>

      

      

    </div>
  </aside>


        
      </div>
    </main>

    <footer id="footer" class="footer">
      <div class="footer-inner">
        <script async src="https://dn-lbstatics.qbox.me/busuanzi/2.3/busuanzi.pure.mini.js"></script>

<div class="powered-by">
<i class="fa fa-user-md"></i><span id="busuanzi_container_site_uv">
  本站访客数:<span id="busuanzi_value_site_uv"></span>
</span>
</div>

<div class="copyright">&copy; <span itemprop="copyrightYear">2018</span>
  <span class="with-love">
    <i class="fa fa-user"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">John Doe</span>

  
</div>


  <div class="powered-by">Powered by <a class="theme-link" target="_blank" href="https://hexo.io">Hexo</a></div>



  <span class="post-meta-divider">|</span>



  <div class="theme-info">Theme &mdash; <a class="theme-link" target="_blank" href="https://github.com/iissnan/hexo-theme-next">NexT.Muse</a> v5.1.4</div>




        







        
      </div>
    </footer>

    
      <div class="back-to-top">
        <i class="fa fa-arrow-up"></i>
        
      </div>
    

    

  </div>

  

<script type="text/javascript">
  if (Object.prototype.toString.call(window.Promise) !== '[object Function]') {
    window.Promise = null;
  }
</script>









  












  
  
    <script type="text/javascript" src="/lib/jquery/index.js?v=2.1.3"></script>
  

  
  
    <script type="text/javascript" src="/lib/fastclick/lib/fastclick.min.js?v=1.0.6"></script>
  

  
  
    <script type="text/javascript" src="/lib/jquery_lazyload/jquery.lazyload.js?v=1.9.7"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>
  

  
  
    <script type="text/javascript" src="/lib/fancybox/source/jquery.fancybox.pack.js?v=2.1.5"></script>
  


  


  <script type="text/javascript" src="/js/src/utils.js?v=5.1.4"></script>

  <script type="text/javascript" src="/js/src/motion.js?v=5.1.4"></script>



  
  

  

  


  <script type="text/javascript" src="/js/src/bootstrap.js?v=5.1.4"></script>



  


  




	





  





  












  





  

  

  

  
  

  

  

  

</body>
</html>
